{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "28c331ef-c544-4562-8e61-57ce89495c39",
   "metadata": {},
   "source": [
    "# **Modelagem molecular por homologia**"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "69d4dcc7-790c-46ce-8acb-cc5991878601",
   "metadata": {},
   "source": [
    "Se observarmos a quantidade de sequências depositadas no **Genbank** (https://www.ncbi.nlm.nih.gov/genbank/statistics/) e compararmos com o número de sequências de proteínas depositadas no **Protein Data Bank** (https://www.rcsb.org), nos deparamos com uma enorme discrepância. Infelizmente, ainda não é possível cristalografar todas as proteínas já sequenciadas por falta de mão de obra, pelo alto custo e por nem sempre a proteína ser passiva de ser cristalografada.\n",
    "\n",
    "Como alternativa, surge a molelagem molecular por homologia, pois como o nome do método já diz, utiliza de um proteína já cristalografada para modelar e prever a estrutura tridimensional da proteína alvo, desde que a proteína alvo e a proteína molde sejam homólogas.\n",
    "\n",
    "Ao falar de homologia entre duas sequências (da proteína alvo a ser modelada e do molde - proteína cristalografada), queremos dizer que ambas compartilham de uma estrutura (alfa-helices, beta-pregueadas, regiões de _loop_) semelhante e função bem conservada evolutivamente (ex.: peptidases que tem por função de romper ligações peptídicas)."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "de123e70-bb72-461e-af03-b5c422cae77e",
   "metadata": {},
   "source": [
    "### Utilizando o _software_ **Modeller** para modelar proteínas"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "215c9b64-3dc2-4006-8436-5b4791fb9211",
   "metadata": {},
   "source": [
    "O software Modeller encontra-se na versão 10.4 e pode ser instalado em sistemas operacionais Linux, Windows e Mac. Sua instalação pelo Anaconda Python é muito simples, mas você precisará realizar um cadastro afim de obter a licença do programa."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "55062858-513d-4f17-b8ba-5ac51d87dd2f",
   "metadata": {},
   "source": [
    "Para instalar basta copiar e colar as seguintes linhas em seu CMD (Windows) ou  terminal de comando (Linux ou Mac) e posteriormente executá-las, uma por vez:\n",
    "\n",
    "```conda config --add canans salilab```\n",
    "\n",
    "```conda install modeller```\n",
    "\n",
    "Lembre que durante a instalação você será solicitado para digitar a chave do produto (_Product Key_)."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "71b4f925-e0e5-4916-885b-385ca372f52d",
   "metadata": {},
   "source": [
    "### Antes de praticar"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5bbcd3d8-072e-4d05-9e5c-b51fdd6195ee",
   "metadata": {},
   "source": [
    "Inicialmente precisamos de uma sequência da proteína alvo para prever ou elucidar a sua estrutura tridimensional. Aqui neste _cookbook_, continuaremos a usar como exemplo a sequência da proteína XPRT (Xantina Fosforribosil Transferase) de _Leishmania donovani_ (ID Genbank: AF170105.1). Esta proteína ainda não possui um modelo cristalográfico depositado no Protein Data Bank (PDB), porém possui proteínas homólogas e que foram depositadas no PDB.\n",
    "\n",
    "A sequência de XPRT encontra-se no arquivo _query_2.fasta_, mas o Modeller não aceita este tipo de arquivo, sendo necessário usar um arquivo em outro formato. Assim, essa sequência será guardada em um arquivo muito semelhante ao fasta, mas no formato _.ali_.\n",
    "\n",
    "A sequência de XPRT está guardada no arquivo _query_2.ali_. Ao observar a estrutura do arquivo _.ali_, iremos perceber na primeira linha a presença de um sinal de maior que (>) indicando o cabeçalho do arquivo, na qual contém o ID e o nome da proteína abreviado.\n",
    "\n",
    "Em seguinda, já na segunda linha, observamos novamente o nome da nossa sequência seguida por pontos e valores iguais a 0,00. O que nos indica que esta sequência não possui estrutura cristalográfica conhecida (afinal, o intuito aqui é elucidar esssa estrutura).\n",
    "\n",
    "Por fim, da terceira linha em diante tem a sequência de aminoácidos da proteína XPRT e ao final dela observamos a presença de um asterísco (_*_) indicando o fim ou o códon de parada.\n",
    "\n",
    "```\n",
    "> AF170105.1; XpRT\n",
    "sequence:XpRT:::::::0.00: 0.00\n",
    "MLPTHSCKGFVDAQGRVFVDGREYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETA\n",
    "EAAEAAEAPISYDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLR\n",
    "FENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGPNKYIV\n",
    "GYGFEVNDRYRDLRHILILRDGEATRYPAKL*\n",
    "```"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "472af8dd-5e75-4889-80ff-640cd485357d",
   "metadata": {},
   "source": [
    "### Primeiro passo: busca de homólogos usando o Blastp"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "b5fc1c23-ac6c-4661-a1cd-24e94506bc96",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<p><!--\n",
      "QBlastInfoBegin\n",
      "\tStatus=READY\n",
      "QBlastInfoEnd\n",
      "--><p>\n",
      "<PRE>\n",
      "BLASTP 2.14.1+\n",
      "Reference: Stephen F. Altschul, Thomas L. Madden, Alejandro\n",
      "A. Schaffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and\n",
      "David J. Lipman (1997), \"Gapped BLAST and PSI-BLAST: a new\n",
      "generation of protein database search programs\", Nucleic\n",
      "Acids Res. 25:3389-3402.\n",
      "\n",
      "\n",
      "Reference for compositional score matrix adjustment: Stephen\n",
      "F. Altschul, John C. Wootton, E. Michael Gertz, Richa\n",
      "Agarwala, Aleksandr Morgulis, Alejandro A. Schaffer, and\n",
      "Yi-Kuo Yu (2005) \"Protein database searches using\n",
      "compositionally adjusted substitution matrices\", FEBS J.\n",
      "272:5101-5109.\n",
      "\n",
      "\n",
      "RID: 6YRWWP1W013\n",
      "\n",
      "\n",
      "Database: PDB protein database\n",
      "           149,544 sequences; 41,078,466 total letters\n",
      "Query= \n",
      "Length=241\n",
      "\n",
      "\n",
      "                                                                   Score     E     Max\n",
      "Sequences producing significant alignments:                       (Bits)  Value  Ident\n",
      "\n",
      "7SCR_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase...  264     2e-88  55%         \n",
      "6AR9_B Crystal structure of hypoxanthine-guanine-xanthine phos...  263     2e-88  55%         \n",
      "6AQO_A Crystal structure of hypoxanthine-guanine-xanthine phos...  259     7e-87  54%         \n",
      "1TC1_A A 1.4 ANGSTROM CRYSTAL STRUCTURE FOR THE HYPOXANTHINE P...  141     4e-41  39%         \n",
      "1P19_A Hypoxanthine Phosphoribosyltransferase from Trypanosoma...  141     4e-41  39%         \n",
      "5JSQ_A Trypanosome brucei Hypoxanthine-guanine phosphoribosylt...  141     4e-41  39%         \n",
      "5EUC_A The role of the C-terminal region on the oligomeric sta...  141     4e-41  39%         \n",
      "1P17_A Chain A, hypoxanthine phosphoribosyltransferase [Trypan...  140     9e-41  38%         \n",
      "1I14_A Chain A, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE...  139     2e-40  38%         \n",
      "1I0L_A Chain A, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE...  139     2e-40  38%         \n",
      "1I0I_A Chain A, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE...  139     3e-40  38%         \n",
      "1I13_A Chain A, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE...  138     5e-40  38%         \n",
      "7CMJ_A Chain A, Hypoxanthine phosphoribosyltransferase [Leishm...  137     1e-39  34%         \n",
      "1PZM_A Crystal structure of HGPRT-ase from Leishmania tarentol...  135     6e-39  33%         \n",
      "3O7M_A 1.98 Angstrom resolution crystal structure of a hypoxan...  106     7e-28  39%         \n",
      "4QYI_B 1.95 Angstrom resolution crystal structure of a hypoxan...  106     7e-28  39%         \n",
      "7SAN_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase...  103     1e-26  34%         \n",
      "5BRN_A Human HGPRT in complex with (S)-HPEPHx, an acyclic nucl...  103     1e-26  34%         \n",
      "6BNJ_A Human hypoxanthine guanine phosphoribosyltransferase in...  102     3e-26  34%         \n",
      "5W8V_A HUMAN HGPRT in complex with [(2-[(guanin-9-yl)methyl]pr...  102     4e-26  34%         \n",
      "1BZY_A Human Hgprtase With Transition State Inhibitor [Homo sa...  102     4e-26  34%         \n",
      "5HIA_A Human hypoxanthine-guanine phosphoribosyltransferase in...  102     4e-26  34%         \n",
      "4KN6_A Crystal structure of human hypoxanthine-guanine phospho...  102     4e-26  34%         \n",
      "4IJQ_A Human hypoxanthine-guanine phosphoribosyltransferase in...  102     4e-26  34%         \n",
      "1D6N_A Chain A, PROTEIN (HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTR...  100     2e-25  33%         \n",
      "1J7J_A Chain A, hypoxanthine phosphoribosyltransferase [Salmon...  96.3    3e-24  36%         \n",
      "2JBH_A Human phosphoribosyl transferase domain containing 1 [H...  96.3    1e-23  35%         \n",
      "8CAG_A Chain A, Hypoxanthine phosphoribosyltransferase [Escher...  92.8    8e-23  36%         \n",
      "5KNR_A E. coli HPRT in complexed with 9-[(N-phosphonoethyl-N-p...  92.8    9e-23  36%         \n",
      "1G9S_A Crystal Structure Of A Complex Between E.Coli Hprt And ...  92.4    1e-22  36%         \n",
      "2VFA_A Crystal structure of a chimera of Plasmodium falciparum...  92.4    3e-22  33%         \n",
      "1HGX_A Hypoxanthine-Guanine-Xanthine Phosphoribosyltransferase...  88.6    3e-21  36%         \n",
      "4PFQ_A Crystal structure of hypoxanthine phosphoribosyltransfe...  85.9    6e-20  33%         \n",
      "4QRI_A 2.35 Angstrom resolution crystal structure of hypoxanth...  85.5    7e-20  32%         \n",
      "1R3U_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase...  80.1    8e-18  30%         \n",
      "2GEB_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase...  79.0    1e-17  30%         \n",
      "5IPF_A Crystal structure of Hypoxanthine-guanine phosphoribosy...  75.9    7e-16  30%         \n",
      "5KNP_A Crystal structure of Mycobacterium tuberculosis hypoxan...  74.7    8e-16  31%         \n",
      "4RHT_A Crystal structures of Mycobacterium tuberculosis 6-oxop...  74.7    8e-16  31%         \n",
      "1CJB_A MALARIAL PURINE PHOSPHORIBOSYLTRANSFERASE [Plasmodium f...  68.6    2e-13  29%         \n",
      "3OZF_A Crystal Structure of Plasmodium falciparum Hypoxanthine...  68.6    3e-13  29%         \n",
      "6IGS_A Crystal structure of HPRT from F. tularensis with Zinc ...  65.9    9e-13  33%         \n",
      "6BO7_A Crystal structure of Plasmodium vivax hypoxanthine guan...  66.2    1e-12  31%         \n",
      "1DBR_A Hypoxanthine Guanine Xanthine [Toxoplasma gondii]           65.5    3e-12  31%         \n",
      "1FSG_A Toxoplasma Gondii Hypoxanthine-Guanine Phosphoribosyltr...  65.5    3e-12  31%         \n",
      "\n",
      "ALIGNMENTS\n",
      ">7SCR_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase [Trypanosoma \n",
      "brucei brucei]\n",
      " 7SCR_B Chain B, Hypoxanthine-guanine phosphoribosyltransferase [Trypanosoma \n",
      "brucei brucei]\n",
      " 7SCR_C Chain C, Hypoxanthine-guanine phosphoribosyltransferase [Trypanosoma \n",
      "brucei brucei]\n",
      " 7SCR_D Chain D, Hypoxanthine-guanine phosphoribosyltransferase [Trypanosoma \n",
      "brucei brucei]\n",
      " 7SCR_E Chain E, Hypoxanthine-guanine phosphoribosyltransferase [Trypanosoma \n",
      "brucei brucei]\n",
      " 7SCR_F Chain F, Hypoxanthine-guanine phosphoribosyltransferase [Trypanosoma \n",
      "brucei brucei]\n",
      "Length=272\n",
      "\n",
      " Score = 264 bits (674),  Expect = 2e-88, Method: Compositional matrix adjust.\n",
      " Identities = 126/230 (55%), Positives = 162/230 (70%), Gaps = 10/230 (4%)\n",
      "\n",
      "Query  12   DAQGRVFVDGREYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAE  71\n",
      "            DA G V VDGR YPMA  +VATE  I  ++K MA T+A  YK+L HRD         T  \n",
      "Sbjct  53   DADGNVTVDGRSYPMAESVVATESTIHRSMKEMAQTLANAYKTLKHRD---------THN  103\n",
      "\n",
      "Query  72   AAEAAEAPISYDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTG  131\n",
      "               +A API+ +NPLIIISVLKGSYIFT+D +RYLGDCGLP+VVDF+R+ SY   TKS+G\n",
      "Sbjct  104  KGNSALAPITDENPLIIISVLKGSYIFTADMVRYLGDCGLPNVVDFIRITSYRGTTKSSG  163\n",
      "\n",
      "Query  132  QISMLAGLRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAAR  191\n",
      "             + +L  LRF  L GKHVLI+ED+ D+GRT++ L + I  +++P S+K  V+V+K    R\n",
      "Sbjct  164  TVQVLDNLRFTELTGKHVLIMEDIADTGRTMKLLVEKIRREYRPASLKVCVLVDKP-GGR  222\n",
      "\n",
      "Query  192  KVDFDPEYFCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPAKL  241\n",
      "             VDF PE+ CL  P +Y+VGYGFEVNDRYR+ RH+ +L+   A RYP+KL\n",
      "Sbjct  223  VVDFKPEFVCLTAPTRYVVGYGFEVNDRYRNYRHVFVLKPEYAKRYPSKL  272\n",
      "\n",
      "\n",
      ">6AR9_B Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei TREU927]\n",
      " 6MXB_A Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with XMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXB_B Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with XMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXB_C Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with XMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXB_D Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with XMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXB_E Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with XMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXB_F Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with XMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXC_A Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with GMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXC_B Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with GMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXC_C Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with GMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXC_D Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with GMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXC_E Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with GMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXC_F Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with GMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXD_A Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with IMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXD_B Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with IMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXD_C Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with IMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      " 6MXD_D Crystal structure of Trypanosoma brucei hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltranferase in complex with IMP [Trypanosoma \n",
      "brucei brucei TREU927]\n",
      "Length=272\n",
      "\n",
      " Score = 263 bits (673),  Expect = 2e-88, Method: Compositional matrix adjust.\n",
      " Identities = 126/230 (55%), Positives = 162/230 (70%), Gaps = 10/230 (4%)\n",
      "\n",
      "Query  12   DAQGRVFVDGREYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAE  71\n",
      "            DA G V VDGR YPMA  +VATE  I  ++K MA T+A  YK+L HRD         T  \n",
      "Sbjct  53   DADGNVTVDGRSYPMAESVVATESTIHRSMKEMAQTLANAYKTLKHRD---------THN  103\n",
      "\n",
      "Query  72   AAEAAEAPISYDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTG  131\n",
      "               +A API+ +NPLIIISVLKGSYIFT+D +RYLGDCGLP+VVDF+R+ SY   TKS+G\n",
      "Sbjct  104  KGNSALAPITDENPLIIISVLKGSYIFTADMVRYLGDCGLPNVVDFIRITSYRGTTKSSG  163\n",
      "\n",
      "Query  132  QISMLAGLRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAAR  191\n",
      "             + +L  LRF  L GKHVLI+ED+ D+GRT++ L + I  +++P S+K  V+V+K    R\n",
      "Sbjct  164  TVQVLDNLRFTELTGKHVLIMEDIADTGRTMKLLVEKIRREYRPASLKVCVLVDKP-GGR  222\n",
      "\n",
      "Query  192  KVDFDPEYFCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPAKL  241\n",
      "             VDF PE+ CL  P +Y+VGYGFEVNDRYR+ RH+ +L+   A RYP+KL\n",
      "Sbjct  223  VVDFKPEFVCLTAPTRYVVGYGFEVNDRYRNYRHVFVLKPEYAKRYPSKL  272\n",
      "\n",
      "\n",
      ">6AQO_A Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with {[(2S)-3-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)propane-1,2-diyl]bis(oxyethane-2,1-diyl)}bis(phosphonic \n",
      "acid) [Trypanosoma brucei brucei TREU927]\n",
      " 6AQO_B Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with {[(2S)-3-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)propane-1,2-diyl]bis(oxyethane-2,1-diyl)}bis(phosphonic \n",
      "acid) [Trypanosoma brucei brucei TREU927]\n",
      " 6AQO_C Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with {[(2S)-3-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)propane-1,2-diyl]bis(oxyethane-2,1-diyl)}bis(phosphonic \n",
      "acid) [Trypanosoma brucei brucei TREU927]\n",
      " 6AQO_D Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with {[(2S)-3-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)propane-1,2-diyl]bis(oxyethane-2,1-diyl)}bis(phosphonic \n",
      "acid) [Trypanosoma brucei brucei TREU927]\n",
      " 6AQO_E Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with {[(2S)-3-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)propane-1,2-diyl]bis(oxyethane-2,1-diyl)}bis(phosphonic \n",
      "acid) [Trypanosoma brucei brucei TREU927]\n",
      " 6AQO_F Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with {[(2S)-3-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)propane-1,2-diyl]bis(oxyethane-2,1-diyl)}bis(phosphonic \n",
      "acid) [Trypanosoma brucei brucei TREU927]\n",
      " 6AR9_A Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei TREU927]\n",
      " 6AR9_C Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei TREU927]\n",
      " 6AR9_D Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei TREU927]\n",
      " 6AR9_E Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei TREU927]\n",
      " 6AR9_F Crystal structure of hypoxanthine-guanine-xanthine phosphorybosyltranferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei TREU927]\n",
      "Length=272\n",
      "\n",
      " Score = 259 bits (663),  Expect = 7e-87, Method: Compositional matrix adjust.\n",
      " Identities = 125/230 (54%), Positives = 161/230 (70%), Gaps = 10/230 (4%)\n",
      "\n",
      "Query  12   DAQGRVFVDGREYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAE  71\n",
      "            DA G V VDGR YPMA  +VATE  I  ++K MA T+A  YK+L HRD         T  \n",
      "Sbjct  53   DADGNVTVDGRSYPMAESVVATESTIHRSMKEMAQTLANAYKTLKHRD---------THN  103\n",
      "\n",
      "Query  72   AAEAAEAPISYDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTG  131\n",
      "               +A API+ +NPLIIISVLKGSYIFT+D +RYLGD GLP+VVDF+R+ SY   TKS+G\n",
      "Sbjct  104  KGNSALAPITDENPLIIISVLKGSYIFTADMVRYLGDXGLPNVVDFIRITSYRGTTKSSG  163\n",
      "\n",
      "Query  132  QISMLAGLRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAAR  191\n",
      "             + +L  LRF  L GKHVLI+ED+ D+GRT++ L + I  +++P S+K  V+V+K    R\n",
      "Sbjct  164  TVQVLDNLRFTELTGKHVLIMEDIADTGRTMKLLVEKIRREYRPASLKVCVLVDKP-GGR  222\n",
      "\n",
      "Query  192  KVDFDPEYFCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPAKL  241\n",
      "             VDF PE+ CL  P +Y+VGYGFEVNDRYR+ RH+ +L+   A RYP+KL\n",
      "Sbjct  223  VVDFKPEFVCLTAPTRYVVGYGFEVNDRYRNYRHVFVLKPEYAKRYPSKL  272\n",
      "\n",
      "\n",
      ">1TC1_A A 1.4 ANGSTROM CRYSTAL STRUCTURE FOR THE HYPOXANTHINE PHOSPHORIBOSYLTRANSFERASE \n",
      "OF TRYPANOSOMA CRUZI [Trypanosoma cruzi]\n",
      " 1TC1_B A 1.4 ANGSTROM CRYSTAL STRUCTURE FOR THE HYPOXANTHINE PHOSPHORIBOSYLTRANSFERASE \n",
      "OF TRYPANOSOMA CRUZI [Trypanosoma cruzi]\n",
      "Length=220\n",
      "\n",
      " Score = 141 bits (356),  Expect = 4e-41, Method: Compositional matrix adjust.\n",
      " Identities = 81/209 (39%), Positives = 121/209 (58%), Gaps = 25/209 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            REY  A  I+ TE+ I+T IK +A  IA DYK    R                       \n",
      "Sbjct  2    REYEFAEKILFTEEEIRTRIKEVAKRIADDYKGKGLR----------------------P  39\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "            Y NPL++ISVLKGS++FT+D  R L D  +P  ++F+ ++SY  G  S+GQ+ ML   R \n",
      "Sbjct  40   YVNPLVLISVLKGSFMFTADLCRALCDFNVPVRMEFICVSSYGEGLTSSGQVRMLLDTR-  98\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++ G HVLIVED+ D+  TL +L      + +P S+KT+V+++K +  R+V F  +Y  \n",
      "Sbjct  99   HSIEGHHVLIVEDIVDTALTLNYLYHMYFTR-RPASLKTVVLLDKRE-GRRVPFSADYVV  156\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN +++GYG + +D YR+LR I++LR\n",
      "Sbjct  157  ANIPNAFVIGYGLDYDDTYRELRDIVVLR  185\n",
      "\n",
      "\n",
      ">1P19_A Hypoxanthine Phosphoribosyltransferase from Trypanosoma cruzi, \n",
      "in complex with the product IMP [Trypanosoma cruzi]\n",
      " 1P19_B Hypoxanthine Phosphoribosyltransferase from Trypanosoma cruzi, \n",
      "in complex with the product IMP [Trypanosoma cruzi]\n",
      " 1P19_C Hypoxanthine Phosphoribosyltransferase from Trypanosoma cruzi, \n",
      "in complex with the product IMP [Trypanosoma cruzi]\n",
      " 1P19_D Hypoxanthine Phosphoribosyltransferase from Trypanosoma cruzi, \n",
      "in complex with the product IMP [Trypanosoma cruzi]\n",
      " 1TC2_A TERNARY SUBSTRATE COMPLEX OF THE HYPOXANTHINE PHOSPHORIBOSYLTRANSFERASE \n",
      "FROM TRYPANOSOMA CRUZI [Trypanosoma cruzi]\n",
      " 1TC2_B TERNARY SUBSTRATE COMPLEX OF THE HYPOXANTHINE PHOSPHORIBOSYLTRANSFERASE \n",
      "FROM TRYPANOSOMA CRUZI [Trypanosoma cruzi]\n",
      "Length=221\n",
      "\n",
      " Score = 141 bits (356),  Expect = 4e-41, Method: Compositional matrix adjust.\n",
      " Identities = 81/209 (39%), Positives = 121/209 (58%), Gaps = 25/209 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            REY  A  I+ TE+ I+T IK +A  IA DYK    R                       \n",
      "Sbjct  3    REYEFAEKILFTEEEIRTRIKEVAKRIADDYKGKGLR----------------------P  40\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "            Y NPL++ISVLKGS++FT+D  R L D  +P  ++F+ ++SY  G  S+GQ+ ML   R \n",
      "Sbjct  41   YVNPLVLISVLKGSFMFTADLCRALCDFNVPVRMEFICVSSYGEGLTSSGQVRMLLDTR-  99\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++ G HVLIVED+ D+  TL +L      + +P S+KT+V+++K +  R+V F  +Y  \n",
      "Sbjct  100  HSIEGHHVLIVEDIVDTALTLNYLYHMYFTR-RPASLKTVVLLDKRE-GRRVPFSADYVV  157\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN +++GYG + +D YR+LR I++LR\n",
      "Sbjct  158  ANIPNAFVIGYGLDYDDTYRELRDIVVLR  186\n",
      "\n",
      "\n",
      ">5JSQ_A Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with a 9-[7-(phosphonoheptyl]guanine [Trypanosoma \n",
      "brucei brucei]\n",
      " 5JSQ_B Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with a 9-[7-(phosphonoheptyl]guanine [Trypanosoma \n",
      "brucei brucei]\n",
      " 5JV5_A Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with Guanosine 5' monophosphate [Trypanosoma \n",
      "brucei brucei]\n",
      " 5JV5_B Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with Guanosine 5' monophosphate [Trypanosoma \n",
      "brucei brucei]\n",
      " 5K51_A Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with a 9-[5-(phosphonoheptyl]hypoxanthine [Trypanosoma \n",
      "brucei brucei]\n",
      " 5K51_B Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with a 9-[5-(phosphonoheptyl]hypoxanthine [Trypanosoma \n",
      "brucei brucei]\n",
      " 5K51_C Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with a 9-[5-(phosphonoheptyl]hypoxanthine [Trypanosoma \n",
      "brucei brucei]\n",
      " 5K51_D Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with a 9-[5-(phosphonoheptyl]hypoxanthine [Trypanosoma \n",
      "brucei brucei]\n",
      " 5KAM_A Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with Inosine 5' monophosphate [Trypanosoma brucei \n",
      "brucei]\n",
      " 5KAM_C Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with Inosine 5' monophosphate [Trypanosoma brucei \n",
      "brucei]\n",
      " 5KAP_A Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with a 9-(4-(phosphonobutil)hypoxanthine [Trypanosoma \n",
      "brucei brucei]\n",
      " 5KAP_B Trypanosome brucei Hypoxanthine-guanine phosphoribosyltranferase \n",
      "in complex with a 9-(4-(phosphonobutil)hypoxanthine [Trypanosoma \n",
      "brucei brucei]\n",
      " 6APS_A Trypanosoma brucei hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [(2-((Guanine-9H-yl)methyl)propane-1,3 \n",
      "diyl)bis(oxy)]bis(methylene))diphosphonic acid [Trypanosoma \n",
      "brucei brucei]\n",
      " 6APS_B Trypanosoma brucei hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [(2-((Guanine-9H-yl)methyl)propane-1,3 \n",
      "diyl)bis(oxy)]bis(methylene))diphosphonic acid [Trypanosoma \n",
      "brucei brucei]\n",
      " 6APT_A Trypanosoma brucei hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with {[(2S)-3-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)propane-1,2-diyl]bis(oxyethane-2,1-diyl)}bis(phosphonic \n",
      "acid) [Trypanosoma brucei brucei]\n",
      " 6APT_B Trypanosoma brucei hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with {[(2S)-3-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)propane-1,2-diyl]bis(oxyethane-2,1-diyl)}bis(phosphonic \n",
      "acid) [Trypanosoma brucei brucei]\n",
      " 6APU_A Crystal structure of Trypanosoma brucei hypoxanthine-guanine \n",
      "phosphoribosyltranferase in complex with (2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl](3-aminopropyl)amino}ethyl)phosphonic \n",
      "acid [Trypanosoma brucei brucei]\n",
      " 6APU_B Crystal structure of Trypanosoma brucei hypoxanthine-guanine \n",
      "phosphoribosyltranferase in complex with (2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl](3-aminopropyl)amino}ethyl)phosphonic \n",
      "acid [Trypanosoma brucei brucei]\n",
      " 6APV_A Trypanosoma brucei hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei]\n",
      " 6APV_B Trypanosoma brucei hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei]\n",
      " 6APV_C Trypanosoma brucei hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei]\n",
      " 6APV_D Trypanosoma brucei hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [(2-{[2-(2-amino-6-oxo-1,6-dihydro-9H-purin-9-yl)ethyl][(E)-2-phosphonoethenyl]amino}ethoxy)methyl]phosphonic \n",
      "acid [Trypanosoma brucei brucei]\n",
      " 6MXG_A Crystal structure of Trypanosoma brucei hypoxanthine-guanine \n",
      "phosphoribosyltranferase in complex with XMP [Trypanosoma brucei \n",
      "brucei]\n",
      " 6MXG_B Crystal structure of Trypanosoma brucei hypoxanthine-guanine \n",
      "phosphoribosyltranferase in complex with XMP [Trypanosoma brucei \n",
      "brucei]\n",
      " 7SB7_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase [Trypanosoma \n",
      "brucei brucei]\n",
      " 7SB7_B Chain B, Hypoxanthine-guanine phosphoribosyltransferase [Trypanosoma \n",
      "brucei brucei]\n",
      "Length=216\n",
      "\n",
      " Score = 141 bits (355),  Expect = 4e-41, Method: Compositional matrix adjust.\n",
      " Identities = 81/209 (39%), Positives = 127/209 (61%), Gaps = 27/209 (13%)\n",
      "\n",
      "Query  23   EYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPISY  82\n",
      "            +Y  A+ ++ TE  + T ++ +A  IA DY + +     L P                  \n",
      "Sbjct  12   KYDFATSVLFTEAELHTRMRGVAQRIADDYSNCN-----LKP-----------------L  49\n",
      "\n",
      "Query  83   DNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLA-GLRF  141\n",
      "            +NPL+I+SVLKGS++FT+D +R LGD G+P  V+F+R +SY   TKS G++ + A GL  \n",
      "Sbjct  50   ENPLVIVSVLKGSFVFTADMVRILGDFGVPTRVEFLRASSYGHDTKSCGRVDVKADGLC-  108\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++RGKHVL++ED+ D+  TLR + D  ++K +P SIKTLV ++K    RK+ F  EY  \n",
      "Sbjct  109  -DIRGKHVLVLEDILDTALTLREVVDS-LKKSEPASIKTLVAIDK-PGGRKIPFTAEYVV  165\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN ++VGYG + +  YR++R ++IL+\n",
      "Sbjct  166  ADVPNVFVVGYGLDYDQSYREVRDVVILK  194\n",
      "\n",
      "\n",
      ">5EUC_A The role of the C-terminal region on the oligomeric state and \n",
      "enzymatic activity of Trypanosoma cruzi hypoxanthine phosphoribosyl \n",
      "transferase [Trypanosoma cruzi strain CL Brener]\n",
      " 5EUC_B The role of the C-terminal region on the oligomeric state and \n",
      "enzymatic activity of Trypanosoma cruzi hypoxanthine phosphoribosyl \n",
      "transferase [Trypanosoma cruzi strain CL Brener]\n",
      " 5EUC_C The role of the C-terminal region on the oligomeric state and \n",
      "enzymatic activity of Trypanosoma cruzi hypoxanthine phosphoribosyl \n",
      "transferase [Trypanosoma cruzi strain CL Brener]\n",
      " 5EUC_D The role of the C-terminal region on the oligomeric state and \n",
      "enzymatic activity of Trypanosoma cruzi hypoxanthine phosphoribosyl \n",
      "transferase [Trypanosoma cruzi strain CL Brener]\n",
      "Length=229\n",
      "\n",
      " Score = 141 bits (356),  Expect = 4e-41, Method: Compositional matrix adjust.\n",
      " Identities = 81/209 (39%), Positives = 121/209 (58%), Gaps = 25/209 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            REY  A  I+ TE+ I+T IK +A  IA DYK    R                       \n",
      "Sbjct  3    REYEFAEKILFTEEEIRTRIKEVAKRIADDYKGKGLR----------------------P  40\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "            Y NPL++ISVLKGS++FT+D  R L D  +P  ++F+ ++SY  G  S+GQ+ ML   R \n",
      "Sbjct  41   YVNPLVLISVLKGSFMFTADLCRALCDFNVPVRMEFICVSSYGEGLTSSGQVRMLLDTR-  99\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++ G HVLIVED+ D+  TL +L      + +P S+KT+V+++K +  R+V F  +Y  \n",
      "Sbjct  100  HSIEGHHVLIVEDIVDTALTLNYLYHMYFTR-RPASLKTVVLLDKRE-GRRVPFSADYVV  157\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN +++GYG + +D YR+LR I++LR\n",
      "Sbjct  158  ANIPNAFVIGYGLDYDDTYRELRDIVVLR  186\n",
      "\n",
      "\n",
      ">1P17_A Chain A, hypoxanthine phosphoribosyltransferase [Trypanosoma \n",
      "cruzi]\n",
      " 1P17_B Chain B, hypoxanthine phosphoribosyltransferase [Trypanosoma \n",
      "cruzi]\n",
      " 1P17_C Chain C, hypoxanthine phosphoribosyltransferase [Trypanosoma \n",
      "cruzi]\n",
      " 1P17_D Chain D, hypoxanthine phosphoribosyltransferase [Trypanosoma \n",
      "cruzi]\n",
      " 1P18_A Chain A, hypoxanthine phosphoribosyltransferase [Trypanosoma \n",
      "cruzi]\n",
      " 1P18_B Chain B, hypoxanthine phosphoribosyltransferase [Trypanosoma \n",
      "cruzi]\n",
      "Length=221\n",
      "\n",
      " Score = 140 bits (353),  Expect = 9e-41, Method: Compositional matrix adjust.\n",
      " Identities = 80/209 (38%), Positives = 121/209 (58%), Gaps = 25/209 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            REY  A  I+ TE+ I+T IK +A  IA DYK    R                       \n",
      "Sbjct  3    REYEFAEKILFTEEEIRTRIKEVAKRIADDYKGKGLR----------------------P  40\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "            Y NPL++ISVL+GS++FT+D  R L D  +P  ++F+ ++SY  G  S+GQ+ ML   R \n",
      "Sbjct  41   YVNPLVLISVLRGSFMFTADLCRALCDFNVPVRMEFICVSSYGEGLTSSGQVRMLLDTR-  99\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++ G HVLIVED+ D+  TL +L      + +P S+KT+V+++K +  R+V F  +Y  \n",
      "Sbjct  100  HSIEGHHVLIVEDIVDTALTLNYLYHMYFTR-RPASLKTVVLLDKRE-GRRVPFSADYVV  157\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN +++GYG + +D YR+LR I++LR\n",
      "Sbjct  158  ANIPNAFVIGYGLDYDDTYRELRDIVVLR  186\n",
      "\n",
      "\n",
      ">1I14_A Chain A, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE [Trypanosoma \n",
      "cruzi]\n",
      " 1I14_B Chain B, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE [Trypanosoma \n",
      "cruzi]\n",
      "Length=221\n",
      "\n",
      " Score = 139 bits (351),  Expect = 2e-40, Method: Compositional matrix adjust.\n",
      " Identities = 80/209 (38%), Positives = 121/209 (58%), Gaps = 25/209 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            REY  A  I+ TE+ I+T IK +A  IA DYK    R                       \n",
      "Sbjct  3    REYEFAEKILFTEEEIRTRIKEVAKRIADDYKGKGLR----------------------P  40\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "            Y NPL++ISVLKGS++FT+D  R L D  +P  ++F+ ++SY  G  S+GQ+ ML   R \n",
      "Sbjct  41   YVNPLVLISVLKGSFMFTADLCRALCDFNVPVRMEFICVSSYGEGLTSSGQVRMLLDTR-  99\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++ G HVLIVED+ ++  TL +L      + +P S+KT+V+++K +  R+V F  +Y  \n",
      "Sbjct  100  HSIEGHHVLIVEDIVETALTLNYLYHMYFTR-RPASLKTVVLLDKRE-GRRVPFSADYVV  157\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN +++GYG + +D YR+LR I++LR\n",
      "Sbjct  158  ANIPNAFVIGYGLDYDDTYRELRDIVVLR  186\n",
      "\n",
      "\n",
      ">1I0L_A Chain A, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE [Trypanosoma \n",
      "cruzi]\n",
      " 1I0L_B Chain B, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE [Trypanosoma \n",
      "cruzi]\n",
      "Length=221\n",
      "\n",
      " Score = 139 bits (351),  Expect = 2e-40, Method: Compositional matrix adjust.\n",
      " Identities = 80/209 (38%), Positives = 121/209 (58%), Gaps = 25/209 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            REY  A  I+ TE+ I+T IK +A  IA DYK    R                       \n",
      "Sbjct  3    REYEFAEKILFTEEEIRTRIKEVAKRIADDYKGKGLR----------------------P  40\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "            Y NPL++ISVLKGS++FT+D  R L D  +P  ++F+ ++SY  G  S+GQ+ ML   R \n",
      "Sbjct  41   YVNPLVLISVLKGSFMFTADLCRALCDFNVPVRMEFICVSSYGEGLTSSGQVRMLLDTR-  99\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++ G HVLIVED+ ++  TL +L      + +P S+KT+V+++K +  R+V F  +Y  \n",
      "Sbjct  100  HSIEGHHVLIVEDIVNTALTLNYLYHMYFTR-RPASLKTVVLLDKRE-GRRVPFSADYVV  157\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN +++GYG + +D YR+LR I++LR\n",
      "Sbjct  158  ANIPNAFVIGYGLDYDDTYRELRDIVVLR  186\n",
      "\n",
      "\n",
      ">1I0I_A Chain A, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE [Trypanosoma \n",
      "cruzi]\n",
      " 1I0I_B Chain B, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE [Trypanosoma \n",
      "cruzi]\n",
      "Length=221\n",
      "\n",
      " Score = 139 bits (350),  Expect = 3e-40, Method: Compositional matrix adjust.\n",
      " Identities = 80/209 (38%), Positives = 120/209 (57%), Gaps = 25/209 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            REY  A  I+ TE+ I+T IK +A  IA DYK    R                       \n",
      "Sbjct  3    REYEFAEKILFTEEEIRTRIKEVAKRIADDYKGKGLR----------------------P  40\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "            Y NPL++ISVLKGS++FT+D  R L D  +P  ++F+ ++SY  G  S+GQ+ ML   R \n",
      "Sbjct  41   YVNPLVLISVLKGSFMFTADLCRALCDFNVPVRMEFICVSSYGEGLTSSGQVRMLLDTR-  99\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++ G HVLIVED+  +  TL +L      + +P S+KT+V+++K +  R+V F  +Y  \n",
      "Sbjct  100  HSIEGHHVLIVEDIVQTALTLNYLYHMYFTR-RPASLKTVVLLDKRE-GRRVPFSADYVV  157\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN +++GYG + +D YR+LR I++LR\n",
      "Sbjct  158  ANIPNAFVIGYGLDYDDTYRELRDIVVLR  186\n",
      "\n",
      "\n",
      ">1I13_A Chain A, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE [Trypanosoma \n",
      "cruzi]\n",
      " 1I13_B Chain B, HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE [Trypanosoma \n",
      "cruzi]\n",
      "Length=221\n",
      "\n",
      " Score = 138 bits (348),  Expect = 5e-40, Method: Compositional matrix adjust.\n",
      " Identities = 80/209 (38%), Positives = 120/209 (57%), Gaps = 25/209 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            REY  A  I+ TE+ I+T IK +A  IA DYK    R                       \n",
      "Sbjct  3    REYEFAEKILFTEEEIRTRIKEVAKRIADDYKGKGLR----------------------P  40\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "            Y NPL++ISVLKGS++FT+D  R L D  +P  ++F+ ++SY  G  S+GQ+ ML   R \n",
      "Sbjct  41   YVNPLVLISVLKGSFMFTADLCRALCDFNVPVRMEFICVSSYGEGLTSSGQVRMLLDTR-  99\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "             ++ G HVLIVED+  +  TL +L      + +P S+KT+V+++K +  R+V F  +Y  \n",
      "Sbjct  100  HSIEGHHVLIVEDIVATALTLNYLYHMYFTR-RPASLKTVVLLDKRE-GRRVPFSADYVV  157\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "               PN +++GYG + +D YR+LR I++LR\n",
      "Sbjct  158  ANIPNAFVIGYGLDYDDTYRELRDIVVLR  186\n",
      "\n",
      "\n",
      ">7CMJ_A Chain A, Hypoxanthine phosphoribosyltransferase [Leishmania donovani \n",
      "BPK282A1]\n",
      " 7CMJ_B Chain B, Hypoxanthine phosphoribosyltransferase [Leishmania donovani \n",
      "BPK282A1]\n",
      "Length=211\n",
      "\n",
      " Score = 137 bits (345),  Expect = 1e-39, Method: Compositional matrix adjust.\n",
      " Identities = 76/225 (34%), Positives = 132/225 (59%), Gaps = 31/225 (14%)\n",
      "\n",
      "Query  6    SCKGFVDAQGRVFVDGREYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPS  65\n",
      "            S  G V  +GR     R YPM++  + T++ +       A  IA+DY+S           \n",
      "Sbjct  7    SPSGPVGDEGR-----RNYPMSAHTLVTQEQVWAATAKCAKKIAEDYRSFK---------  52\n",
      "\n",
      "Query  66   TAETAEAAEAAEAPISYDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNS  125\n",
      "                          ++ DNPL ++ VLKGS+IFT+D  R+L D G+P  V+F+  +SY +\n",
      "Sbjct  53   --------------LTTDNPLYLLCVLKGSFIFTADLARFLADEGVPVKVEFICASSYGT  98\n",
      "\n",
      "Query  126  GTKSTGQISMLAGLRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVN  185\n",
      "            G +++GQ+ ML  +R +++  +H+LIVED+ DS  TL++L  +++ K +P S+KT+V+++\n",
      "Sbjct  99   GVETSGQVRMLLDVR-DSVENRHILIVEDIVDSAITLQYLMRFMLAK-KPASLKTVVLLD  156\n",
      "\n",
      "Query  186  KEQAARKVDFDPEYFCLAGPNKYIVGYGFEVNDRYRDLRHILILR  230\n",
      "            K  + RKV+   +Y  +  P+ +++GYG +  + YR+LR I +L+\n",
      "Sbjct  157  K-PSGRKVEVLVDYPVITIPHAFVIGYGMDYAESYRELRDICVLK  200\n",
      "\n",
      "\n",
      ">1PZM_A Crystal structure of HGPRT-ase from Leishmania tarentolae in \n",
      "complex with GMP [Leishmania tarentolae]\n",
      " 1PZM_B Crystal structure of HGPRT-ase from Leishmania tarentolae in \n",
      "complex with GMP [Leishmania tarentolae]\n",
      "Length=211\n",
      "\n",
      " Score = 135 bits (340),  Expect = 6e-39, Method: Compositional matrix adjust.\n",
      " Identities = 73/220 (33%), Positives = 127/220 (58%), Gaps = 26/220 (12%)\n",
      "\n",
      "Query  22   REYPMASGIVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPIS  81\n",
      "            R YPM++  + T++ +       A  IA DYK                          ++\n",
      "Sbjct  18   RNYPMSARTLVTQEQVWAATAKCAKKIAADYKDFH-----------------------LT  54\n",
      "\n",
      "Query  82   YDNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRF  141\n",
      "             DNPL ++ VLKGS+IFT+D  R+L D G+P  V+F+  +SY SG +++GQ+ ML  +R \n",
      "Sbjct  55   ADNPLYLLCVLKGSFIFTADLARFLADEGVPVKVEFICASSYGSGVETSGQVRMLLDVR-  113\n",
      "\n",
      "Query  142  ENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFC  201\n",
      "            +++  +H+++VED+ DS  TL++L  +++ K +P S+KT+V+++K  + RKVD   +Y  \n",
      "Sbjct  114  DSVENRHIMLVEDIVDSAITLQYLMRFMLAK-KPASLKTVVLLDK-PSGRKVDVLVDYPV  171\n",
      "\n",
      "Query  202  LAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPAKL  241\n",
      "            +  P  +++GYG +  + YR+LR I +L+     +  +KL\n",
      "Sbjct  172  ITIPRAFVIGYGMDFAESYRELRDICVLKKEYYEKAESKL  211\n",
      "\n",
      "\n",
      ">3O7M_A 1.98 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' [Bacillus anthracis str. 'Ames Ancestor']\n",
      " 3O7M_B 1.98 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' [Bacillus anthracis str. 'Ames Ancestor']\n",
      " 3O7M_C 1.98 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' [Bacillus anthracis str. 'Ames Ancestor']\n",
      " 3O7M_D 1.98 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' [Bacillus anthracis str. 'Ames Ancestor']\n",
      " 4QYI_A 1.95 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' with HEPES molecule in the active \n",
      "site [Bacillus anthracis]\n",
      "Length=186\n",
      "\n",
      " Score = 106 bits (264),  Expect = 7e-28, Method: Compositional matrix adjust.\n",
      " Identities = 55/142 (39%), Positives = 95/142 (67%), Gaps = 8/142 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLR  145\n",
      "            +++I+VLKGS++F +D IR++ +      +DF+  +SY + T++TG++ +L  +   N+ \n",
      "Sbjct  38   IVVIAVLKGSFVFAADLIRHIKN---DVTIDFISASSYGNQTETTGKVKLLKDIDV-NIT  93\n",
      "\n",
      "Query  146  GKHVLIVEDVCDSGRTLRFLRD-YIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAG  204\n",
      "            GK+V++VED+ DSG TL FL+D + M K  PK++K   +++K +  RKVD   EY     \n",
      "Sbjct  94   GKNVIVVEDIIDSGLTLHFLKDHFFMHK--PKALKFCTLLDKPE-RRKVDLTAEYVGFQI  150\n",
      "\n",
      "Query  205  PNKYIVGYGFEVNDRYRDLRHI  226\n",
      "            P+++IVGYG +  ++YR+L  I\n",
      "Sbjct  151  PDEFIVGYGIDCAEKYRNLPFI  172\n",
      "\n",
      "\n",
      ">4QYI_B 1.95 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' with HEPES molecule in the active \n",
      "site [Bacillus anthracis]\n",
      " 4QYI_C 1.95 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' with HEPES molecule in the active \n",
      "site [Bacillus anthracis]\n",
      " 4QYI_D 1.95 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' with HEPES molecule in the active \n",
      "site [Bacillus anthracis]\n",
      " 4QYI_E 1.95 Angstrom resolution crystal structure of a hypoxanthine-guanine \n",
      "phosphoribosyltransferase (hpt-2) from Bacillus anthracis \n",
      "str. 'Ames Ancestor' with HEPES molecule in the active \n",
      "site [Bacillus anthracis]\n",
      "Length=186\n",
      "\n",
      " Score = 106 bits (264),  Expect = 7e-28, Method: Compositional matrix adjust.\n",
      " Identities = 55/142 (39%), Positives = 95/142 (67%), Gaps = 8/142 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLR  145\n",
      "            +++I+VLKGS++F +D IR++ +      +DF+  +SY + T++TG++ +L  +   N+ \n",
      "Sbjct  38   IVVIAVLKGSFVFAADLIRHIKN---DVTIDFISASSYGNQTETTGKVKLLKDIDV-NIT  93\n",
      "\n",
      "Query  146  GKHVLIVEDVCDSGRTLRFLRD-YIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAG  204\n",
      "            GK+V++VED+ DSG TL FL+D + M K  PK++K   +++K +  RKVD   EY     \n",
      "Sbjct  94   GKNVIVVEDIIDSGLTLHFLKDHFFMHK--PKALKFCTLLDKPE-RRKVDLTAEYVGFQI  150\n",
      "\n",
      "Query  205  PNKYIVGYGFEVNDRYRDLRHI  226\n",
      "            P+++IVGYG +  ++YR+L  I\n",
      "Sbjct  151  PDEFIVGYGIDXAEKYRNLPFI  172\n",
      "\n",
      "\n",
      ">7SAN_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase [Homo \n",
      "sapiens]\n",
      " 7SAN_B Chain B, Hypoxanthine-guanine phosphoribosyltransferase [Homo \n",
      "sapiens]\n",
      " 7SAN_C Chain C, Hypoxanthine-guanine phosphoribosyltransferase [Homo \n",
      "sapiens]\n",
      " 7SAN_D Chain D, Hypoxanthine-guanine phosphoribosyltransferase [Homo \n",
      "sapiens]\n",
      "Length=217\n",
      "\n",
      " Score = 103 bits (257),  Expect = 1e-26, Method: Compositional matrix adjust.\n",
      " Identities = 54/160 (34%), Positives = 94/160 (59%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY +  +STG I ++ G \n",
      "Sbjct  61   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-ANDQSTGDIKVIGGD  119\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  120  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  177\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  178  VGFEIPDKFVVGYALDYNEYFRDLNHVAVISETGKAKYKA  217\n",
      "\n",
      "\n",
      ">5BRN_A Human HGPRT in complex with (S)-HPEPHx, an acyclic nucleoside \n",
      "phosphonate [Homo sapiens]\n",
      " 5BRN_B Human HGPRT in complex with (S)-HPEPHx, an acyclic nucleoside \n",
      "phosphonate [Homo sapiens]\n",
      " 5BRN_C Human HGPRT in complex with (S)-HPEPHx, an acyclic nucleoside \n",
      "phosphonate [Homo sapiens]\n",
      " 5BRN_D Human HGPRT in complex with (S)-HPEPHx, an acyclic nucleoside \n",
      "phosphonate [Homo sapiens]\n",
      " 5BSK_A Human HGPRT in complex with (S)-HPEPG, an acyclic nucleoside \n",
      "phosphonate [Homo sapiens]\n",
      " 5BSK_B Human HGPRT in complex with (S)-HPEPG, an acyclic nucleoside \n",
      "phosphonate [Homo sapiens]\n",
      " 5BSK_C Human HGPRT in complex with (S)-HPEPG, an acyclic nucleoside \n",
      "phosphonate [Homo sapiens]\n",
      " 5BSK_D Human HGPRT in complex with (S)-HPEPG, an acyclic nucleoside \n",
      "phosphonate [Homo sapiens]\n",
      "Length=218\n",
      "\n",
      " Score = 103 bits (257),  Expect = 1e-26, Method: Compositional matrix adjust.\n",
      " Identities = 54/160 (34%), Positives = 94/160 (59%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY +  +STG I ++ G \n",
      "Sbjct  62   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-ANDQSTGDIKVIGGD  120\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  121  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  178\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  179  VGFEIPDKFVVGYALDYNEYFRDLNHVAVISETGKAKYKA  218\n",
      "\n",
      "\n",
      ">6BNJ_A Human hypoxanthine guanine phosphoribosyltransferase in complex \n",
      "with [3R,4R]-4-guanin-9-yl-3-((R)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Homo sapiens]\n",
      " 6BNJ_B Human hypoxanthine guanine phosphoribosyltransferase in complex \n",
      "with [3R,4R]-4-guanin-9-yl-3-((R)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Homo sapiens]\n",
      " 6BNJ_C Human hypoxanthine guanine phosphoribosyltransferase in complex \n",
      "with [3R,4R]-4-guanin-9-yl-3-((R)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Homo sapiens]\n",
      " 6BNJ_D Human hypoxanthine guanine phosphoribosyltransferase in complex \n",
      "with [3R,4R]-4-guanin-9-yl-3-((R)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Homo sapiens]\n",
      "Length=218\n",
      "\n",
      " Score = 102 bits (255),  Expect = 3e-26, Method: Compositional matrix adjust.\n",
      " Identities = 54/160 (34%), Positives = 93/160 (58%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY    +STG I ++ G \n",
      "Sbjct  62   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-CNDQSTGDIKVIGGD  120\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  121  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  178\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  179  VGFEIPDKFVVGYALDYNEYFRDLNHVAVISETGKAKYKA  218\n",
      "\n",
      "\n",
      ">5W8V_A HUMAN HGPRT in complex with [(2-[(guanin-9-yl)methyl]propane-1,3-diyl)bis(oxy)]bis(methylene)diphosphonic \n",
      "acid [Homo sapiens]\n",
      " 5W8V_B HUMAN HGPRT in complex with [(2-[(guanin-9-yl)methyl]propane-1,3-diyl)bis(oxy)]bis(methylene)diphosphonic \n",
      "acid [Homo sapiens]\n",
      " 5W8V_C HUMAN HGPRT in complex with [(2-[(guanin-9-yl)methyl]propane-1,3-diyl)bis(oxy)]bis(methylene)diphosphonic \n",
      "acid [Homo sapiens]\n",
      " 5W8V_D HUMAN HGPRT in complex with [(2-[(guanin-9-yl)methyl]propane-1,3-diyl)bis(oxy)]bis(methylene)diphosphonic \n",
      "acid [Homo sapiens]\n",
      "Length=215\n",
      "\n",
      " Score = 102 bits (254),  Expect = 4e-26, Method: Compositional matrix adjust.\n",
      " Identities = 54/160 (34%), Positives = 93/160 (58%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY    +STG I ++ G \n",
      "Sbjct  59   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-CNDQSTGDIKVIGGD  117\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  118  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  175\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  176  VGFEIPDKFVVGYALDYNEYFRDLNHVCVISETGKAKYKA  215\n",
      "\n",
      "\n",
      ">1BZY_A Human Hgprtase With Transition State Inhibitor [Homo sapiens]\n",
      " 1BZY_B Human Hgprtase With Transition State Inhibitor [Homo sapiens]\n",
      " 1BZY_C Human Hgprtase With Transition State Inhibitor [Homo sapiens]\n",
      " 1BZY_D Human Hgprtase With Transition State Inhibitor [Homo sapiens]\n",
      " 1HMP_A THE CRYSTAL STRUCTURE OF HUMAN HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "WITH BOUND GMP [Homo sapiens]\n",
      " 1HMP_B THE CRYSTAL STRUCTURE OF HUMAN HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "WITH BOUND GMP [Homo sapiens]\n",
      " 1Z7G_A Free human HGPRT [Homo sapiens]\n",
      " 1Z7G_B Free human HGPRT [Homo sapiens]\n",
      " 1Z7G_C Free human HGPRT [Homo sapiens]\n",
      " 1Z7G_D Free human HGPRT [Homo sapiens]\n",
      " 3GEP_A Human hypoxanthine guanine phosphoribosyltranserfase in complex \n",
      "with (S)-9-(3-hydroxy-2-phosphonylmethoxypropyl)guanine [Homo \n",
      "sapiens]\n",
      " 3GEP_B Human hypoxanthine guanine phosphoribosyltranserfase in complex \n",
      "with (S)-9-(3-hydroxy-2-phosphonylmethoxypropyl)guanine [Homo \n",
      "sapiens]\n",
      " 3GGC_A Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with 9-(2-phosphonoethoxyethyl)hypoxanthine [Homo sapiens]\n",
      " 3GGC_B Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with 9-(2-phosphonoethoxyethyl)hypoxanthine [Homo sapiens]\n",
      " 3GGJ_A Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with 9-(2-phosphonoethoxyethyl)guanine [Homo sapiens]\n",
      " 3GGJ_B Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with 9-(2-phosphonoethoxyethyl)guanine [Homo sapiens]\n",
      " 4RAB_A Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAB_B Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAB_C Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAB_D Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAC_A Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAC_B Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAC_C Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAC_D Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAD_A Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAD_B Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAD_C Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAD_D Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAD_E Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAD_F Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAD_G Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAD_H Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAN_A Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAN_B Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAN_C Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAN_D Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAO_A Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAO_B Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAO_C Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAO_D Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAQ_A Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAQ_B Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAQ_C Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      " 4RAQ_D Aza-acyclic nucleoside phosphonates containing a second phosphonate \n",
      "group as inhibitors of the human, Plasmodium falciparum \n",
      "and vivax 6-oxopurine phosphoribosyltransferases and their \n",
      "pro-drugs as antimalarial agents [Homo sapiens]\n",
      "Length=217\n",
      "\n",
      " Score = 102 bits (254),  Expect = 4e-26, Method: Compositional matrix adjust.\n",
      " Identities = 54/160 (34%), Positives = 93/160 (58%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY    +STG I ++ G \n",
      "Sbjct  61   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-CNDQSTGDIKVIGGD  119\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  120  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  177\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  178  VGFEIPDKFVVGYALDYNEYFRDLNHVCVISETGKAKYKA  217\n",
      "\n",
      "\n",
      ">5HIA_A Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Homo sapiens]\n",
      " 5HIA_B Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Homo sapiens]\n",
      " 5HIA_C Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Homo sapiens]\n",
      " 5HIA_D Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Homo sapiens]\n",
      "Length=224\n",
      "\n",
      " Score = 102 bits (255),  Expect = 4e-26, Method: Compositional matrix adjust.\n",
      " Identities = 54/160 (34%), Positives = 93/160 (58%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY    +STG I ++ G \n",
      "Sbjct  68   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-CNDQSTGDIKVIGGD  126\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  127  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  184\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  185  VGFEIPDKFVVGYALDYNEYFRDLNHVAVISETGKAKYKA  224\n",
      "\n",
      "\n",
      ">4KN6_A Crystal structure of human hypoxanthine-guanine phosphoribosyltransferase \n",
      "in complex with 6-fluoro-3-hydroxy-2-pyrazinecarboxamide \n",
      "(T-705) ribose-5'-monophosphate [Homo sapiens]\n",
      "Length=216\n",
      "\n",
      " Score = 102 bits (254),  Expect = 4e-26, Method: Compositional matrix adjust.\n",
      " Identities = 54/160 (34%), Positives = 93/160 (58%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY    +STG I ++ G \n",
      "Sbjct  60   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-CNDQSTGDIKVIGGD  118\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  119  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  176\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  177  VGFEIPDKFVVGYALDYNEYFRDLNHVCVISETGKAKYKA  216\n",
      "\n",
      "\n",
      ">4IJQ_A Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with [(2-((Guanine-9H-yl)methyl)propane-1,3-diyl)bis(oxy)]bis(methylene))diphosphonic \n",
      "acid [Homo sapiens]\n",
      " 4IJQ_B Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with [(2-((Guanine-9H-yl)methyl)propane-1,3-diyl)bis(oxy)]bis(methylene))diphosphonic \n",
      "acid [Homo sapiens]\n",
      " 4IJQ_C Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with [(2-((Guanine-9H-yl)methyl)propane-1,3-diyl)bis(oxy)]bis(methylene))diphosphonic \n",
      "acid [Homo sapiens]\n",
      " 4IJQ_D Human hypoxanthine-guanine phosphoribosyltransferase in complex \n",
      "with [(2-((Guanine-9H-yl)methyl)propane-1,3-diyl)bis(oxy)]bis(methylene))diphosphonic \n",
      "acid [Homo sapiens]\n",
      "Length=223\n",
      "\n",
      " Score = 102 bits (254),  Expect = 4e-26, Method: Compositional matrix adjust.\n",
      " Identities = 54/160 (34%), Positives = 93/160 (58%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY    +STG I ++ G \n",
      "Sbjct  67   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-CNDQSTGDIKVIGGD  125\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  126  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  183\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  184  VGFEIPDKFVVGYALDYNEYFRDLNHVCVISETGKAKYKA  223\n",
      "\n",
      "\n",
      ">1D6N_A Chain A, PROTEIN (HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE) \n",
      "[Homo sapiens]\n",
      " 1D6N_B Chain B, PROTEIN (HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE) \n",
      "[Homo sapiens]\n",
      "Length=214\n",
      "\n",
      " Score = 100 bits (249),  Expect = 2e-25, Method: Compositional matrix adjust.\n",
      " Identities = 53/160 (33%), Positives = 92/160 (58%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VL G Y F +D + Y+       D  +P  VDF+RL SY    +STG I ++ G \n",
      "Sbjct  58   IVALCVLAGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLGSY-CNDQSTGDIKVIGGD  116\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   ++ K +  R V + P++\n",
      "Sbjct  117  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVASLLVK-RTPRSVGYKPDF  174\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "                 P+K++VGY  + N+ +RDL H+ ++ +    +Y A\n",
      "Sbjct  175  VGFEIPDKFVVGYALDYNEYFRDLNHVCVISETGKAKYKA  214\n",
      "\n",
      "\n",
      ">1J7J_A Chain A, hypoxanthine phosphoribosyltransferase [Salmonella enterica \n",
      "subsp. enterica serovar Typhimurium]\n",
      " 1J7J_B Chain B, hypoxanthine phosphoribosyltransferase [Salmonella enterica \n",
      "subsp. enterica serovar Typhimurium]\n",
      "Length=178\n",
      "\n",
      " Score = 96.3 bits (238),  Expect = 3e-24, Method: Compositional matrix adjust.\n",
      " Identities = 52/143 (36%), Positives = 91/143 (64%), Gaps = 6/143 (4%)\n",
      "\n",
      "Query  84   NPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFEN  143\n",
      "            + ++++ +L+GS++F +D  R   +  +PH VDF+  +SY SG  +T  + +L  L  E+\n",
      "Sbjct  34   SEMVLVGLLRGSFMFMADLCR---EVQVPHEVDFMTASSYGSGMSTTRDVKILKDLD-ED  89\n",
      "\n",
      "Query  144  LRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLA  203\n",
      "            +RGK VLIVED+ DSG TL  +R+ I+   +PKS+    +++K  + R+VD   E+   +\n",
      "Sbjct  90   IRGKDVLIVEDIIDSGNTLSKVRE-ILGLREPKSLAICTLLDK-PSRREVDVPVEFVGFS  147\n",
      "\n",
      "Query  204  GPNKYIVGYGFEVNDRYRDLRHI  226\n",
      "             P++++VGYG +   RYR L ++\n",
      "Sbjct  148  IPDEFVVGYGIDYAQRYRHLPYV  170\n",
      "\n",
      "\n",
      ">2JBH_A Human phosphoribosyl transferase domain containing 1 [Homo sapiens]\n",
      " 2JBH_B Human phosphoribosyl transferase domain containing 1 [Homo sapiens]\n",
      "Length=225\n",
      "\n",
      " Score = 96.3 bits (238),  Expect = 1e-23, Method: Compositional matrix adjust.\n",
      " Identities = 56/158 (35%), Positives = 91/158 (58%), Gaps = 9/158 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++++ VLKG Y F +D + +L       D  +   VDF+RL SY +  +S G++ ++ G \n",
      "Sbjct  69   IMVLCVLKGGYKFXADLVEHLKNISRNSDRFVSMKVDFIRLKSYRN-DQSMGEMQIIGGD  127\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVEDV  +GRT++ L   I EK++P  IK   ++ K + +R   F P+Y\n",
      "Sbjct  128  DLSTLAGKNVLIVEDVVGTGRTMKALLSNI-EKYKPNMIKVASLLVK-RTSRSDGFRPDY  185\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRY  237\n",
      "                 PN ++VGY  + N+ +RDL HI ++ +    +Y\n",
      "Sbjct  186  AGFEIPNLFVVGYALDYNEYFRDLNHICVINEHGKEKY  223\n",
      "\n",
      "\n",
      ">8CAG_A Chain A, Hypoxanthine phosphoribosyltransferase [Escherichia \n",
      "coli]\n",
      " 8CAG_B Chain B, Hypoxanthine phosphoribosyltransferase [Escherichia \n",
      "coli]\n",
      "Length=176\n",
      "\n",
      " Score = 92.8 bits (229),  Expect = 8e-23, Method: Compositional matrix adjust.\n",
      " Identities = 51/141 (36%), Positives = 89/141 (63%), Gaps = 6/141 (4%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLR  145\n",
      "            ++++ +L+GS++F +D  R   +  + H VDF+  +SY SG  +T  + +L  L  E++R\n",
      "Sbjct  36   MVLVGLLRGSFMFMADLCR---EVQVSHEVDFMTASSYGSGMSTTRDVKILKDLD-EDIR  91\n",
      "\n",
      "Query  146  GKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGP  205\n",
      "            GK VLIVED+ DSG TL  +R+ I+   +PKS+    +++K  + R+V+   E+   + P\n",
      "Sbjct  92   GKDVLIVEDIIDSGNTLSKVRE-ILSLREPKSLAICTLLDK-PSRREVNVPVEFIGFSIP  149\n",
      "\n",
      "Query  206  NKYIVGYGFEVNDRYRDLRHI  226\n",
      "            ++++VGYG +   RYR L +I\n",
      "Sbjct  150  DEFVVGYGIDYAQRYRHLPYI  170\n",
      "\n",
      "\n",
      ">5KNR_A E. coli HPRT in complexed with 9-[(N-phosphonoethyl-N-phosphonoethoxyethyl)-2-aminoethyl]-guanine \n",
      "[Escherichia coli]\n",
      " 5KNR_B E. coli HPRT in complexed with 9-[(N-phosphonoethyl-N-phosphonoethoxyethyl)-2-aminoethyl]-guanine \n",
      "[Escherichia coli]\n",
      " 5KNS_A E coli hypoxanthine guanine phosphoribosyltransferase in complexed \n",
      "with 9-[(N-phosphonoethyl-N-phosphonoethoxyethyl)-2-aminoethyl]hypoxanthine \n",
      "[Escherichia coli]\n",
      " 5KNS_B E coli hypoxanthine guanine phosphoribosyltransferase in complexed \n",
      "with 9-[(N-phosphonoethyl-N-phosphonoethoxyethyl)-2-aminoethyl]hypoxanthine \n",
      "[Escherichia coli]\n",
      " 5KNT_A Crystal structure of E. coli hypoxanthine phosphoribosyltransferase \n",
      "in complexed with 2-((2,3-Dihydroxypropyl)(2-(hypoxanthin-9-yl)ethyl)amino)ethylphosphonic \n",
      "acid [Escherichia coli]\n",
      " 5KNT_B Crystal structure of E. coli hypoxanthine phosphoribosyltransferase \n",
      "in complexed with 2-((2,3-Dihydroxypropyl)(2-(hypoxanthin-9-yl)ethyl)amino)ethylphosphonic \n",
      "acid [Escherichia coli]\n",
      " 5KNU_A Crystal structure of E. coli hypoxanthine phosphoribosyltransferase \n",
      "in complexed with 9-[N,N-(Bis-3-phosphonopropyl)aminomethyl]-9-deazahypoxanthine \n",
      "[Escherichia coli]\n",
      " 5KNU_B Crystal structure of E. coli hypoxanthine phosphoribosyltransferase \n",
      "in complexed with 9-[N,N-(Bis-3-phosphonopropyl)aminomethyl]-9-deazahypoxanthine \n",
      "[Escherichia coli]\n",
      " 5KNV_A E coli hypoxanthine guanine phosphoribosyltransferase in complexed \n",
      "with 9-[(N-Phosphonoethyl-N-phosphonobutyl)-2-aminoethyl]-hypoxanthine \n",
      "[Escherichia coli]\n",
      " 5KNV_B E coli hypoxanthine guanine phosphoribosyltransferase in complexed \n",
      "with 9-[(N-Phosphonoethyl-N-phosphonobutyl)-2-aminoethyl]-hypoxanthine \n",
      "[Escherichia coli]\n",
      " 5KNX_A Crystal structure of E. coli hypoxanthine phosphoribosyltransferase \n",
      "in complexed with {[(2-[(Hypoxanthin-9H-yl)methyl]propane-1,3-diyl)bis(oxy)]bis- \n",
      "(methylene)}diphosphonic Acid [Escherichia \n",
      "coli]\n",
      " 5KNX_B Crystal structure of E. coli hypoxanthine phosphoribosyltransferase \n",
      "in complexed with {[(2-[(Hypoxanthin-9H-yl)methyl]propane-1,3-diyl)bis(oxy)]bis- \n",
      "(methylene)}diphosphonic Acid [Escherichia \n",
      "coli]\n",
      "Length=182\n",
      "\n",
      " Score = 92.8 bits (229),  Expect = 9e-23, Method: Compositional matrix adjust.\n",
      " Identities = 51/141 (36%), Positives = 89/141 (63%), Gaps = 6/141 (4%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLR  145\n",
      "            ++++ +L+GS++F +D  R   +  + H VDF+  +SY SG  +T  + +L  L  E++R\n",
      "Sbjct  40   MVLVGLLRGSFMFMADLCR---EVQVSHEVDFMTASSYGSGMSTTRDVKILKDLD-EDIR  95\n",
      "\n",
      "Query  146  GKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGP  205\n",
      "            GK VLIVED+ DSG TL  +R+ I+   +PKS+    +++K  + R+V+   E+   + P\n",
      "Sbjct  96   GKDVLIVEDIIDSGNTLSKVRE-ILSLREPKSLAICTLLDK-PSRREVNVPVEFIGFSIP  153\n",
      "\n",
      "Query  206  NKYIVGYGFEVNDRYRDLRHI  226\n",
      "            ++++VGYG +   RYR L +I\n",
      "Sbjct  154  DEFVVGYGIDYAQRYRHLPYI  174\n",
      "\n",
      "\n",
      ">1G9S_A Crystal Structure Of A Complex Between E.Coli Hprt And Imp [Escherichia \n",
      "coli]\n",
      " 1G9S_B Crystal Structure Of A Complex Between E.Coli Hprt And Imp [Escherichia \n",
      "coli]\n",
      " 1G9T_A Crystal Structure Of E.Coli Hprt-Gmp Complex [Escherichia coli]\n",
      " 1G9T_B Crystal Structure Of E.Coli Hprt-Gmp Complex [Escherichia coli]\n",
      " 1GRV_A Hypoxanthine Phosphoribosyltransferase from E. coli [Escherichia \n",
      "coli]\n",
      " 1GRV_B Hypoxanthine Phosphoribosyltransferase from E. coli [Escherichia \n",
      "coli]\n",
      "Length=182\n",
      "\n",
      " Score = 92.4 bits (228),  Expect = 1e-22, Method: Compositional matrix adjust.\n",
      " Identities = 51/141 (36%), Positives = 89/141 (63%), Gaps = 6/141 (4%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLR  145\n",
      "            ++++ +L+GS++F +D  R   +  + H VDF+  +SY SG  +T  + +L  L  E++R\n",
      "Sbjct  40   MVLVGLLRGSFMFMADLCR---EVQVSHEVDFMTASSYGSGMSTTRDLKILKDLD-EDIR  95\n",
      "\n",
      "Query  146  GKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGP  205\n",
      "            GK VLIVED+ DSG TL  +R+ I+   +PKS+    +++K  + R+V+   E+   + P\n",
      "Sbjct  96   GKDVLIVEDIIDSGNTLSKVRE-ILSLREPKSLAICTLLDK-PSRREVNVPVEFIGFSIP  153\n",
      "\n",
      "Query  206  NKYIVGYGFEVNDRYRDLRHI  226\n",
      "            ++++VGYG +   RYR L +I\n",
      "Sbjct  154  DEFVVGYGIDYAQRYRHLPYI  174\n",
      "\n",
      "\n",
      ">2VFA_A Crystal structure of a chimera of Plasmodium falciparum and human \n",
      "hypoxanthine-guanine phosphoribosyl transferases [Plasmodium \n",
      "falciparum]\n",
      " 2VFA_B Crystal structure of a chimera of Plasmodium falciparum and human \n",
      "hypoxanthine-guanine phosphoribosyl transferases [Plasmodium \n",
      "falciparum]\n",
      "Length=229\n",
      "\n",
      " Score = 92.4 bits (228),  Expect = 3e-22, Method: Compositional matrix adjust.\n",
      " Identities = 52/160 (33%), Positives = 87/160 (54%), Gaps = 9/160 (6%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYL------GDCGLPHVVDFVRLASYNSGTKSTGQISMLAGL  139\n",
      "            ++ + VLKG Y F +D + Y+       D  +P  VDF+RL SY    +STG I ++ G \n",
      "Sbjct  70   IVALCVLKGGYKFFADLLDYIKALNRNSDRSIPMTVDFIRLKSY-CNDQSTGDIKVIGGD  128\n",
      "\n",
      "Query  140  RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEY  199\n",
      "                L GK+VLIVED+ D+G+T++ L   ++ ++ PK +K   +  K        F  ++\n",
      "Sbjct  129  DLSTLTGKNVLIVEDIIDTGKTMQTLLS-LVRQYNPKMVKVACLFIKRTPLWN-GFKADF  186\n",
      "\n",
      "Query  200  FCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "               + P+ ++VGY  + N+ +RDL H  ++ D    +Y A\n",
      "Sbjct  187  VGFSIPDHFVVGYSLDYNEIFRDLDHCCLVNDEGKKKYKA  226\n",
      "\n",
      "\n",
      ">1HGX_A Hypoxanthine-Guanine-Xanthine Phosphoribosyltransferase (Hgxprtase) \n",
      "[Tritrichomonas foetus]\n",
      " 1HGX_B Hypoxanthine-Guanine-Xanthine Phosphoribosyltransferase (Hgxprtase) \n",
      "[Tritrichomonas foetus]\n",
      "Length=183\n",
      "\n",
      " Score = 88.6 bits (218),  Expect = 3e-21, Method: Compositional matrix adjust.\n",
      " Identities = 59/164 (36%), Positives = 97/164 (59%), Gaps = 11/164 (7%)\n",
      "\n",
      "Query  75   AAEAPISYD--NPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQ  132\n",
      "            AAE    Y+  NP ++I VL G+  F +D +++L     P   D++  +SY SGTKSTG \n",
      "Sbjct  28   AAELTEFYEDKNP-VMICVLTGAVFFYTDLLKHLDFQLEP---DYIICSSY-SGTKSTGN  82\n",
      "\n",
      "Query  133  ISMLAGLRFENLRGKHVLIVEDVCDSGRTL-RFLRDYIMEKFQPKSIKTLVMVNKEQAAR  191\n",
      "            +++   L+  N+ G+HVL+VED+ D+G T+ + L +  M K  P S+K   + +K+   +\n",
      "Sbjct  83   LTISKDLK-TNIEGRHVLVVEDIIDTGLTMYQLLNNLQMRK--PASLKVCTLCDKDIGKK  139\n",
      "\n",
      "Query  192  KVDFDPEYFCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEAT  235\n",
      "              D   +Y      N+YI+GYGF+ +++YR+L  I IL++   T\n",
      "Sbjct  140  AYDVPIDYCGFVVENRYIIGYGFDFHNKYRNLPVIGILKESVYT  183\n",
      "\n",
      "\n",
      ">4PFQ_A Crystal structure of hypoxanthine phosphoribosyltransferase from \n",
      "Brachybacterium faecium DSM 4810, NYSGRC Target 029763. \n",
      "[Brachybacterium faecium DSM 4810]\n",
      " 4PFQ_B Crystal structure of hypoxanthine phosphoribosyltransferase from \n",
      "Brachybacterium faecium DSM 4810, NYSGRC Target 029763. \n",
      "[Brachybacterium faecium DSM 4810]\n",
      " 4PFQ_C Crystal structure of hypoxanthine phosphoribosyltransferase from \n",
      "Brachybacterium faecium DSM 4810, NYSGRC Target 029763. \n",
      "[Brachybacterium faecium DSM 4810]\n",
      " 4PFQ_D Crystal structure of hypoxanthine phosphoribosyltransferase from \n",
      "Brachybacterium faecium DSM 4810, NYSGRC Target 029763. \n",
      "[Brachybacterium faecium DSM 4810]\n",
      " 4PFQ_E Crystal structure of hypoxanthine phosphoribosyltransferase from \n",
      "Brachybacterium faecium DSM 4810, NYSGRC Target 029763. \n",
      "[Brachybacterium faecium DSM 4810]\n",
      " 4PFQ_F Crystal structure of hypoxanthine phosphoribosyltransferase from \n",
      "Brachybacterium faecium DSM 4810, NYSGRC Target 029763. \n",
      "[Brachybacterium faecium DSM 4810]\n",
      " 4PFQ_G Crystal structure of hypoxanthine phosphoribosyltransferase from \n",
      "Brachybacterium faecium DSM 4810, NYSGRC Target 029763. \n",
      "[Brachybacterium faecium DSM 4810]\n",
      " 4PFQ_H Crystal structure of hypoxanthine phosphoribosyltransferase from \n",
      "Brachybacterium faecium DSM 4810, NYSGRC Target 029763. \n",
      "[Brachybacterium faecium DSM 4810]\n",
      "Length=207\n",
      "\n",
      " Score = 85.9 bits (211),  Expect = 6e-20, Method: Compositional matrix adjust.\n",
      " Identities = 49/147 (33%), Positives = 88/147 (60%), Gaps = 6/147 (4%)\n",
      "\n",
      "Query  83   DNPLIIISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFE  142\n",
      "            + P +++ VL+G+    +D  R +    L    D+  ++SY SGTKS+G + +L  L   \n",
      "Sbjct  60   EEPPVLVGVLRGAVXVXADLARQID---LKVEXDWXAVSSYGSGTKSSGVVRILKDLS-G  115\n",
      "\n",
      "Query  143  NLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCL  202\n",
      "            ++  ++VLIVED+ DSG TL++L   +  +  PKS++   ++ K  AAR VD D +Y   \n",
      "Sbjct  116  DITDRNVLIVEDIIDSGLTLKWLLSNLRSR-GPKSVEVAALLRKPDAAR-VDIDVKYIGF  173\n",
      "\n",
      "Query  203  AGPNKYIVGYGFEVNDRYRDLRHILIL  229\n",
      "              P+++++GYG +  + YR+L ++ +L\n",
      "Sbjct  174  DIPSEFVIGYGLDYAENYRNLPYVGVL  200\n",
      "\n",
      "\n",
      ">4QRI_A 2.35 Angstrom resolution crystal structure of hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltransferase from Leptospira interrogans \n",
      "serovar Copenhageni str. Fiocruz L1-130 [Leptospira \n",
      "interrogans serovar Copenhageni str. Fiocruz L1-130]\n",
      " 4QRI_B 2.35 Angstrom resolution crystal structure of hypoxanthine-guanine-xanthine \n",
      "phosphoribosyltransferase from Leptospira interrogans \n",
      "serovar Copenhageni str. Fiocruz L1-130 [Leptospira \n",
      "interrogans serovar Copenhageni str. Fiocruz L1-130]\n",
      "Length=207\n",
      "\n",
      " Score = 85.5 bits (210),  Expect = 7e-20, Method: Compositional matrix adjust.\n",
      " Identities = 66/208 (32%), Positives = 109/208 (52%), Gaps = 38/208 (18%)\n",
      "\n",
      "Query  32   ATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPISYDNPLIIISV  91\n",
      "            + ED+ Q  +K++A  I++DYK L                            NP I I V\n",
      "Sbjct  38   SREDISQ-KVKSLALQISEDYKKL----------------------------NP-IFICV  67\n",
      "\n",
      "Query  92   LKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLRGKHVLI  151\n",
      "            LKG   F +D  R   +      ++FV+  SY SGT STG+I +L  +  + L  +HV+I\n",
      "Sbjct  68   LKGGVYFFTDLTR---EIPFSVEINFVQARSY-SGTVSTGKIELLKDIDID-LSDRHVII  122\n",
      "\n",
      "Query  152  VEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGPNKYIVG  211\n",
      "            VED+ D+G TL++L  +I  +  P S++ + ++ KE+    ++F  +Y     P++++VG\n",
      "Sbjct  123  VEDILDTGFTLQYLVRHIFTR-NPASLEIVTLLLKERKD-TLEFPVKYIGWRIPDEFLVG  180\n",
      "\n",
      "Query  212  YGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "            YG + + RYR+L  I +L  GE  ++P \n",
      "Sbjct  181  YGLDFDGRYRNLPDIHVLEPGEP-QFPV  207\n",
      "\n",
      "\n",
      ">1R3U_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase [Caldanaerobacter \n",
      "subterraneus subsp. tengcongensis]\n",
      " 1R3U_B Chain B, Hypoxanthine-guanine phosphoribosyltransferase [Caldanaerobacter \n",
      "subterraneus subsp. tengcongensis]\n",
      " 1YFZ_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase [Caldanaerobacter \n",
      "subterraneus subsp. tengcongensis MB4]\n",
      " 1YFZ_B Chain B, Hypoxanthine-guanine phosphoribosyltransferase [Caldanaerobacter \n",
      "subterraneus subsp. tengcongensis MB4]\n",
      "Length=205\n",
      "\n",
      " Score = 80.1 bits (196),  Expect = 8e-18, Method: Compositional matrix adjust.\n",
      " Identities = 60/201 (30%), Positives = 107/201 (53%), Gaps = 35/201 (17%)\n",
      "\n",
      "Query  30   IVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPISYDNPLIII  89\n",
      "            I+ TE+ ++  +K +   I +DY+                                L++I\n",
      "Sbjct  35   ILITEEQLKAKVKELGEMITRDYEG-----------------------------KDLVLI  65\n",
      "\n",
      "Query  90   SVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLRGKHV  149\n",
      "             VLKG+ +F S   R +    LP  +DF+ ++SY S TKS+G + ++      ++ GK V\n",
      "Sbjct  66   GVLKGAIMFMSGLSRAID---LPLSIDFLAVSSYGSSTKSSGIVKIIKDHDI-DIEGKDV  121\n",
      "\n",
      "Query  150  LIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGPNKYI  209\n",
      "            LIVED+ DSG TL +LR+ ++ + +P+S+K   +++K +  R+ D   +Y     P+K++\n",
      "Sbjct  122  LIVEDIIDSGLTLAYLRETLLGR-KPRSLKICTILDKPER-READVKVDYCGFKIPDKFV  179\n",
      "\n",
      "Query  210  VGYGFEVNDRYRDLRHILILR  230\n",
      "            VGYG +  ++YR+L  I +L+\n",
      "Sbjct  180  VGYGLDYAEKYRNLPFIGVLK  200\n",
      "\n",
      "\n",
      ">2GEB_A Chain A, Hypoxanthine-guanine phosphoribosyltransferase [Caldanaerobacter \n",
      "subterraneus subsp. tengcongensis]\n",
      "Length=185\n",
      "\n",
      " Score = 79.0 bits (193),  Expect = 1e-17, Method: Compositional matrix adjust.\n",
      " Identities = 60/201 (30%), Positives = 107/201 (53%), Gaps = 35/201 (17%)\n",
      "\n",
      "Query  30   IVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPISYDNPLIII  89\n",
      "            I+ TE+ ++  +K +   I +DY+                                L++I\n",
      "Sbjct  15   ILITEEQLKAKVKELGEMITRDYEG-----------------------------KDLVLI  45\n",
      "\n",
      "Query  90   SVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLRGKHV  149\n",
      "             VLKG+ +F S   R +    LP  +DF+ ++SY S TKS+G + ++      ++ GK V\n",
      "Sbjct  46   GVLKGAIMFMSGLSRAID---LPLSIDFLAVSSYGSSTKSSGIVKIIKDHDI-DIEGKDV  101\n",
      "\n",
      "Query  150  LIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGPNKYI  209\n",
      "            LIVED+ DSG TL +LR+ ++ + +P+S+K   +++K +  R+ D   +Y     P+K++\n",
      "Sbjct  102  LIVEDIIDSGLTLAYLRETLLGR-KPRSLKICTILDKPER-READVKVDYCGFKIPDKFV  159\n",
      "\n",
      "Query  210  VGYGFEVNDRYRDLRHILILR  230\n",
      "            VGYG +  ++YR+L  I +L+\n",
      "Sbjct  160  VGYGIDYAEKYRNLPFIGVLK  180\n",
      "\n",
      "\n",
      ">5IPF_A Crystal structure of Hypoxanthine-guanine phosphoribosyltransferase \n",
      "from Schistosoma mansoni in complex with IMP [Schistosoma \n",
      "mansoni]\n",
      " 5IPF_B Crystal structure of Hypoxanthine-guanine phosphoribosyltransferase \n",
      "from Schistosoma mansoni in complex with IMP [Schistosoma \n",
      "mansoni]\n",
      " 5IPF_C Crystal structure of Hypoxanthine-guanine phosphoribosyltransferase \n",
      "from Schistosoma mansoni in complex with IMP [Schistosoma \n",
      "mansoni]\n",
      " 5IPF_D Crystal structure of Hypoxanthine-guanine phosphoribosyltransferase \n",
      "from Schistosoma mansoni in complex with IMP [Schistosoma \n",
      "mansoni]\n",
      "Length=250\n",
      "\n",
      " Score = 75.9 bits (185),  Expect = 7e-16, Method: Compositional matrix adjust.\n",
      " Identities = 53/178 (30%), Positives = 89/178 (50%), Gaps = 16/178 (9%)\n",
      "\n",
      "Query  61   RLSPSTAETAEAAEAAEAPISYDNPLIIISVLKGSYIFTSDFIRYLG------DCGLPHV  114\n",
      "            RL   +    +  EA  A       + ++ VLKG + F +D +  L          LP  \n",
      "Sbjct  67   RLEKMSMNIVDYYEACNA-----TSITLMCVLKGGFKFLADLVDGLERTVRARGIVLPMS  121\n",
      "\n",
      "Query  115  VDFVRLASYNSGTKSTGQISMLAGL-RFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKF  173\n",
      "            V+FVR+ SY +       I  L GL      + K+VL+VED+ D+G+T+  L  + ++  \n",
      "Sbjct  122  VEFVRVKSYVNDVSIHEPI--LTGLGDPSEYKDKNVLVVEDIIDTGKTITKLISH-LDSL  178\n",
      "\n",
      "Query  174  QPKSIKTLVMVNKEQAARKVDFDPEYFCLAGPNKYIVGYGFEVNDRYRDLRHILILRD  231\n",
      "              KS+K   ++ K  + R  D+ P++     PN+++VGY  + ND +RDL HI ++ +\n",
      "Sbjct  179  STKSVKVASLLVKRTSPRN-DYRPDFVGFEVPNRFVVGYALDYNDNFRDLHHICVINE  235\n",
      "\n",
      "\n",
      ">5KNP_A Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with [3S,4R]-(4-(Hypoxanthin-9-yl)pyrrolidin-3-yl)-oxymethanephosphonic \n",
      "acid \n",
      "[Mycobacterium tuberculosis H37Rv]\n",
      " 5KNP_B Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with [3S,4R]-(4-(Hypoxanthin-9-yl)pyrrolidin-3-yl)-oxymethanephosphonic \n",
      "acid \n",
      "[Mycobacterium tuberculosis H37Rv]\n",
      " 5KNP_C Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with [3S,4R]-(4-(Hypoxanthin-9-yl)pyrrolidin-3-yl)-oxymethanephosphonic \n",
      "acid \n",
      "[Mycobacterium tuberculosis H37Rv]\n",
      " 5KNP_D Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with [3S,4R]-(4-(Hypoxanthin-9-yl)pyrrolidin-3-yl)-oxymethanephosphonic \n",
      "acid \n",
      "[Mycobacterium tuberculosis H37Rv]\n",
      " 5KNQ_A Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with [3S,4R]-(4-(Guanin-9-yl)pyrrolidin-3-yl)oxymethanephosphonic \n",
      "acid and \n",
      "pyrophosphate [Mycobacterium tuberculosis H37Rv]\n",
      " 5KNQ_B Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with [3S,4R]-(4-(Guanin-9-yl)pyrrolidin-3-yl)oxymethanephosphonic \n",
      "acid and \n",
      "pyrophosphate [Mycobacterium tuberculosis H37Rv]\n",
      " 5KNQ_C Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with [3S,4R]-(4-(Guanin-9-yl)pyrrolidin-3-yl)oxymethanephosphonic \n",
      "acid and \n",
      "pyrophosphate [Mycobacterium tuberculosis H37Rv]\n",
      " 5KNQ_D Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with [3S,4R]-(4-(Guanin-9-yl)pyrrolidin-3-yl)oxymethanephosphonic \n",
      "acid and \n",
      "pyrophosphate [Mycobacterium tuberculosis H37Rv]\n",
      " 5KNY_A Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with (3-((3R,4R)-3-(Guanin-9-yl)-4-((S)-2-hydroxy-2-phosphonoethoxy)pyrrolidin-1-yl)-3-oxopropyl)phosphonic \n",
      "acid [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 5KNY_B Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with (3-((3R,4R)-3-(Guanin-9-yl)-4-((S)-2-hydroxy-2-phosphonoethoxy)pyrrolidin-1-yl)-3-oxopropyl)phosphonic \n",
      "acid [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 5KNY_C Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with (3-((3R,4R)-3-(Guanin-9-yl)-4-((S)-2-hydroxy-2-phosphonoethoxy)pyrrolidin-1-yl)-3-oxopropyl)phosphonic \n",
      "acid [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 5KNY_D Crystal structure of Mycobacterium tuberculosis hypoxanthine \n",
      "guanine phosphoribosyltransferase in complex with (3-((3R,4R)-3-(Guanin-9-yl)-4-((S)-2-hydroxy-2-phosphonoethoxy)pyrrolidin-1-yl)-3-oxopropyl)phosphonic \n",
      "acid [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      "Length=207\n",
      "\n",
      " Score = 74.7 bits (182),  Expect = 8e-16, Method: Compositional matrix adjust.\n",
      " Identities = 61/197 (31%), Positives = 101/197 (51%), Gaps = 29/197 (15%)\n",
      "\n",
      "Query  30   IVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPISYDNPLIII  89\n",
      "            ++ T + IQ  I  +   I  DY+ LS          A T +              L++I\n",
      "Sbjct  31   VLLTAEQIQARIAELGEQIGNDYRELS----------ATTGQ-------------DLLLI  67\n",
      "\n",
      "Query  90   SVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLRGKHV  149\n",
      "            +VLKG+ +F +D  R +    +P   +F+ ++SY S T S+G + +L  L   ++ G+ V\n",
      "Sbjct  68   TVLKGAVLFVTDLARAIP---VPTQFEFMAVSSYGSSTSSSGVVRILKDLD-RDIHGRDV  123\n",
      "\n",
      "Query  150  LIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGPNKYI  209\n",
      "            LIVEDV DSG TL +L   +  +  P+S++   ++ K  A    + +  Y     PN ++\n",
      "Sbjct  124  LIVEDVVDSGLTLSWLSRNLTSR-NPRSLRVCTLLRKPDAVHA-NVEIAYVGFDIPNDFV  181\n",
      "\n",
      "Query  210  VGYGFEVNDRYRDLRHI  226\n",
      "            VGYG + ++RYRDL +I\n",
      "Sbjct  182  VGYGLDYDERYRDLSYI  198\n",
      "\n",
      "\n",
      ">4RHT_A Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHT_B Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHT_C Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHT_D Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHU_A Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHU_B Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHU_C Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHU_D Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHU_E Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHU_F Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Rv]\n",
      " 4RHX_A Structures of Mycobacterium tuberculosis 6-oxopurine phosphoribosyltransferase \n",
      "which is a potential target for drug development \n",
      "against this disease [Mycobacterium tuberculosis H37Ra]\n",
      " 4RHX_B Structures of Mycobacterium tuberculosis 6-oxopurine phosphoribosyltransferase \n",
      "which is a potential target for drug development \n",
      "against this disease [Mycobacterium tuberculosis H37Ra]\n",
      " 4RHX_C Structures of Mycobacterium tuberculosis 6-oxopurine phosphoribosyltransferase \n",
      "which is a potential target for drug development \n",
      "against this disease [Mycobacterium tuberculosis H37Ra]\n",
      " 4RHX_D Structures of Mycobacterium tuberculosis 6-oxopurine phosphoribosyltransferase \n",
      "which is a potential target for drug development \n",
      "against this disease [Mycobacterium tuberculosis H37Ra]\n",
      " 4RHY_A Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Ra]\n",
      " 4RHY_B Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Ra]\n",
      " 4RHY_C Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Ra]\n",
      " 4RHY_D Crystal structures of Mycobacterium tuberculosis 6-oxopurine \n",
      "phosphoribosyltransferase which is a potential target for drug \n",
      "development against this disease [Mycobacterium tuberculosis \n",
      "H37Ra]\n",
      "Length=201\n",
      "\n",
      " Score = 74.7 bits (182),  Expect = 8e-16, Method: Compositional matrix adjust.\n",
      " Identities = 61/197 (31%), Positives = 101/197 (51%), Gaps = 29/197 (15%)\n",
      "\n",
      "Query  30   IVATEDVIQTNIKAMAHTIAKDYKSLSHRDARLSPSTAETAEAAEAAEAPISYDNPLIII  89\n",
      "            ++ T + IQ  I  +   I  DY+ LS          A T +              L++I\n",
      "Sbjct  25   VLLTAEQIQARIAELGEQIGNDYRELS----------ATTGQ-------------DLLLI  61\n",
      "\n",
      "Query  90   SVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISMLAGLRFENLRGKHV  149\n",
      "            +VLKG+ +F +D  R +    +P   +F+ ++SY S T S+G + +L  L   ++ G+ V\n",
      "Sbjct  62   TVLKGAVLFVTDLARAIP---VPTQFEFMAVSSYGSSTSSSGVVRILKDLD-RDIHGRDV  117\n",
      "\n",
      "Query  150  LIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGPNKYI  209\n",
      "            LIVEDV DSG TL +L   +  +  P+S++   ++ K  A    + +  Y     PN ++\n",
      "Sbjct  118  LIVEDVVDSGLTLSWLSRNLTSR-NPRSLRVCTLLRKPDAVHA-NVEIAYVGFDIPNDFV  175\n",
      "\n",
      "Query  210  VGYGFEVNDRYRDLRHI  226\n",
      "            VGYG + ++RYRDL +I\n",
      "Sbjct  176  VGYGLDYDERYRDLSYI  192\n",
      "\n",
      "\n",
      ">1CJB_A MALARIAL PURINE PHOSPHORIBOSYLTRANSFERASE [Plasmodium falciparum]\n",
      " 1CJB_B MALARIAL PURINE PHOSPHORIBOSYLTRANSFERASE [Plasmodium falciparum]\n",
      " 1CJB_C MALARIAL PURINE PHOSPHORIBOSYLTRANSFERASE [Plasmodium falciparum]\n",
      " 1CJB_D MALARIAL PURINE PHOSPHORIBOSYLTRANSFERASE [Plasmodium falciparum]\n",
      "Length=231\n",
      "\n",
      " Score = 68.6 bits (166),  Expect = 2e-13, Method: Compositional matrix adjust.\n",
      " Identities = 47/161 (29%), Positives = 84/161 (52%), Gaps = 13/161 (8%)\n",
      "\n",
      "Query  88   IISVLKGSYIFTSDFIRYLGDCGLPHVVD---------FVRLASYNSGTKSTGQISMLAG  138\n",
      "            I+ +LKGS  F +  +++L        V+         +VR+ SY    +STG + +++ \n",
      "Sbjct  72   ILCLLKGSRGFFTALLKHLSRIHNYSAVETSKPLFGEHYVRVKSY-CNDQSTGTLEIVSE  130\n",
      "\n",
      "Query  139  LRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPE  198\n",
      "                 L+GKHVLIVED+ D+G+TL    +Y ++KF+ K++    +  K        F  +\n",
      "Sbjct  131  -DLSCLKGKHVLIVEDIIDTGKTLVKFCEY-LKKFEIKTVAIACLFIKRTPLWN-GFKAD  187\n",
      "\n",
      "Query  199  YFCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "            +   + P+ ++VGY  + N+ +RDL H  ++ D    +Y A\n",
      "Sbjct  188  FVGFSIPDHFVVGYSLDYNEIFRDLDHCCLVNDEGKKKYKA  228\n",
      "\n",
      "\n",
      ">3OZF_A Crystal Structure of Plasmodium falciparum Hypoxanthine-Guanine-Xanthine \n",
      "Phosphoribosyltransferase in complex with hypoxanthine \n",
      "[Plasmodium falciparum FCR-3/Gambia]\n",
      " 3OZF_B Crystal Structure of Plasmodium falciparum Hypoxanthine-Guanine-Xanthine \n",
      "Phosphoribosyltransferase in complex with hypoxanthine \n",
      "[Plasmodium falciparum FCR-3/Gambia]\n",
      " 3OZF_C Crystal Structure of Plasmodium falciparum Hypoxanthine-Guanine-Xanthine \n",
      "Phosphoribosyltransferase in complex with hypoxanthine \n",
      "[Plasmodium falciparum FCR-3/Gambia]\n",
      " 3OZF_D Crystal Structure of Plasmodium falciparum Hypoxanthine-Guanine-Xanthine \n",
      "Phosphoribosyltransferase in complex with hypoxanthine \n",
      "[Plasmodium falciparum FCR-3/Gambia]\n",
      " 3OZG_A Crystal Structure of Plasmodium falciparum Hypoxanthine-Guanine-Xanthine \n",
      "Phosphoribosyltransferase in complex with S-SerMe-ImmH \n",
      "phosphonate [Plasmodium falciparum FCR-3/Gambia]\n",
      " 3OZG_B Crystal Structure of Plasmodium falciparum Hypoxanthine-Guanine-Xanthine \n",
      "Phosphoribosyltransferase in complex with S-SerMe-ImmH \n",
      "phosphonate [Plasmodium falciparum FCR-3/Gambia]\n",
      " 3OZG_C Crystal Structure of Plasmodium falciparum Hypoxanthine-Guanine-Xanthine \n",
      "Phosphoribosyltransferase in complex with S-SerMe-ImmH \n",
      "phosphonate [Plasmodium falciparum FCR-3/Gambia]\n",
      " 3OZG_D Crystal Structure of Plasmodium falciparum Hypoxanthine-Guanine-Xanthine \n",
      "Phosphoribosyltransferase in complex with S-SerMe-ImmH \n",
      "phosphonate [Plasmodium falciparum FCR-3/Gambia]\n",
      " 7TUX_A Chain A, Hypoxanthine-guanine-xanthine phosphoribosyltransferase \n",
      "[Plasmodium falciparum]\n",
      " 7TUX_B Chain B, Hypoxanthine-guanine-xanthine phosphoribosyltransferase \n",
      "[Plasmodium falciparum]\n",
      " 7TUX_C Chain C, Hypoxanthine-guanine-xanthine phosphoribosyltransferase \n",
      "[Plasmodium falciparum]\n",
      " 7TUX_D Chain D, Hypoxanthine-guanine-xanthine phosphoribosyltransferase \n",
      "[Plasmodium falciparum]\n",
      "Length=250\n",
      "\n",
      " Score = 68.6 bits (166),  Expect = 3e-13, Method: Compositional matrix adjust.\n",
      " Identities = 47/161 (29%), Positives = 84/161 (52%), Gaps = 13/161 (8%)\n",
      "\n",
      "Query  88   IISVLKGSYIFTSDFIRYLGDCGLPHVVD---------FVRLASYNSGTKSTGQISMLAG  138\n",
      "            I+ +LKGS  F +  +++L        V+         +VR+ SY    +STG + +++ \n",
      "Sbjct  91   ILCLLKGSRGFFTALLKHLSRIHNYSAVETSKPLFGEHYVRVKSY-CNDQSTGTLEIVSE  149\n",
      "\n",
      "Query  139  LRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPE  198\n",
      "                 L+GKHVLIVED+ D+G+TL    +Y ++KF+ K++    +  K        F  +\n",
      "Sbjct  150  -DLSCLKGKHVLIVEDIIDTGKTLVKFCEY-LKKFEIKTVAIACLFIKRTPLWN-GFKAD  206\n",
      "\n",
      "Query  199  YFCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRYPA  239\n",
      "            +   + P+ ++VGY  + N+ +RDL H  ++ D    +Y A\n",
      "Sbjct  207  FVGFSIPDHFVVGYSLDYNEIFRDLDHCCLVNDEGKKKYKA  247\n",
      "\n",
      "\n",
      ">6IGS_A Crystal structure of HPRT from F. tularensis with Zinc [Francisella \n",
      "tularensis]\n",
      " 6IGS_B Crystal structure of HPRT from F. tularensis with Zinc [Francisella \n",
      "tularensis]\n",
      " 6IGS_C Crystal structure of HPRT from F. tularensis with Zinc [Francisella \n",
      "tularensis]\n",
      " 6IGS_D Crystal structure of HPRT from F. tularensis with Zinc [Francisella \n",
      "tularensis]\n",
      "Length=177\n",
      "\n",
      " Score = 65.9 bits (159),  Expect = 9e-13, Method: Compositional matrix adjust.\n",
      " Identities = 47/144 (33%), Positives = 88/144 (61%), Gaps = 6/144 (4%)\n",
      "\n",
      "Query  88   IISVLKGSYIFTSDFIRYLGDCGLPHVVDFVRLASYNSGTKSTGQISML-AGLRFENLRG  146\n",
      "            ++ VLKGS++F +D +R L    +     F+  +SY S TKS+G +++    L+ E ++ \n",
      "Sbjct  39   LVCVLKGSFMFFADLVRKLR---IDLRTQFITASSYGSSTKSSGTVTLTETSLKEEYVKD  95\n",
      "\n",
      "Query  147  KHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPEYFCLAGPN  206\n",
      "            K+++I+ED+ D+G T   L + I  K+ PK++K   ++ K  A  + D   +Y C    +\n",
      "Sbjct  96   KNIIIIEDIVDTGHTYHKLIEGI-GKYNPKTLKFATLLFKP-ARLERDVKLDYVCFEIED  153\n",
      "\n",
      "Query  207  KYIVGYGFEVNDRYRDLRHILILR  230\n",
      "            K+IVGYG + +++YR+L +I +++\n",
      "Sbjct  154  KFIVGYGLDFDEKYRELPYIGLIK  177\n",
      "\n",
      "\n",
      ">6BO7_A Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_B Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_C Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_D Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_E Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_F Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_G Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_H Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_I Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_J Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_K Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      " 6BO7_L Crystal structure of Plasmodium vivax hypoxanthine guanine phosphoribosyltransferase \n",
      "in complex with [3R,4R]-4-guanin-9-yl-3-((S)-2-hydroxy-2-phosphonoethyl)oxy-1-N-(phosphonopropionyl)pyrrolidine \n",
      "[Plasmodium vivax]\n",
      "Length=238\n",
      "\n",
      " Score = 66.2 bits (160),  Expect = 1e-12, Method: Compositional matrix adjust.\n",
      " Identities = 47/153 (31%), Positives = 82/153 (54%), Gaps = 13/153 (8%)\n",
      "\n",
      "Query  88   IISVLKGSYIFTSDFIRYLG------DCGLP---HVVDFVRLASYNSGTKSTGQISMLAG  138\n",
      "            +I +LKGS  F S  ++YL           P   +V  +VR+ SY    +S  +I +++ \n",
      "Sbjct  77   VICLLKGSRGFFSALLKYLNRIHNYSSTESPKHLYVEHYVRVKSY-CNDQSLDRIEIVSE  135\n",
      "\n",
      "Query  139  LRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFDPE  198\n",
      "                 L+ KHVLIVED+ D+G+TL    +Y ++KF+ K+I    +  K        F  +\n",
      "Sbjct  136  -DLSCLKDKHVLIVEDIIDTGKTLLKFCEY-LKKFEVKTIAITCLFIKRTPLWN-GFKAD  192\n",
      "\n",
      "Query  199  YFCLAGPNKYIVGYGFEVNDRYRDLRHILILRD  231\n",
      "            +   + P+ ++VGY  + N+++RDL H+ ++ D\n",
      "Sbjct  193  FVGFSIPDAFVVGYSLDYNEKFRDLDHLCLVND  225\n",
      "\n",
      "\n",
      ">1DBR_A Hypoxanthine Guanine Xanthine [Toxoplasma gondii]\n",
      " 1DBR_B Hypoxanthine Guanine Xanthine [Toxoplasma gondii]\n",
      " 1DBR_C Hypoxanthine Guanine Xanthine [Toxoplasma gondii]\n",
      " 1DBR_D Hypoxanthine Guanine Xanthine [Toxoplasma gondii]\n",
      "Length=231\n",
      "\n",
      " Score = 65.5 bits (158),  Expect = 3e-12, Method: Compositional matrix adjust.\n",
      " Identities = 50/161 (31%), Positives = 83/161 (52%), Gaps = 13/161 (8%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYLG--------DCGLPHVVD-FVRLASYNSGTKSTGQISML  136\n",
      "            L II +LKGS  F +  I YL         +  +P   + +VRL SY +   STGQ+++L\n",
      "Sbjct  72   LHIICILKGSRGFFNLLIDYLATIQKYSGRESSVPPFFEHYVRLKSYQNDN-STGQLTVL  130\n",
      "\n",
      "Query  137  AGLRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFD  196\n",
      "            +       R KHVLIVED+ D+G TL    +  ++   PKS++   +V K +  R     \n",
      "Sbjct  131  SD-DLSIFRDKHVLIVEDIVDTGFTLTEFGER-LKAVGPKSMRIATLVEK-RTDRSNSLK  187\n",
      "\n",
      "Query  197  PEYFCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRY  237\n",
      "             ++   +  + +IVG  ++ N+ +RD  H+ +L D    ++\n",
      "Sbjct  188  GDFVGFSIEDVWIVGCCYDFNEMFRDFDHVAVLSDAARKKF  228\n",
      "\n",
      "\n",
      ">1FSG_A Toxoplasma Gondii Hypoxanthine-Guanine Phosphoribosyltransferase \n",
      "Complexed With 9-Deazaguanine, Alpha-D-5-Phosphoribosyl-1- \n",
      "Pyrophosphate (Prpp) And Two Mg2+ Ions [Toxoplasma gondii \n",
      "RH]\n",
      " 1FSG_C Toxoplasma Gondii Hypoxanthine-Guanine Phosphoribosyltransferase \n",
      "Complexed With 9-Deazaguanine, Alpha-D-5-Phosphoribosyl-1- \n",
      "Pyrophosphate (Prpp) And Two Mg2+ Ions [Toxoplasma gondii \n",
      "RH]\n",
      " 1QK3_A TOXOPLASMA GONDII HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "GMP COMPLEX [Toxoplasma gondii RH]\n",
      " 1QK3_B TOXOPLASMA GONDII HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "GMP COMPLEX [Toxoplasma gondii RH]\n",
      " 1QK3_C TOXOPLASMA GONDII HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "GMP COMPLEX [Toxoplasma gondii RH]\n",
      " 1QK3_D TOXOPLASMA GONDII HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "GMP COMPLEX [Toxoplasma gondii RH]\n",
      " 1QK4_A TOXOPLASMA GONDII HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "IMP COMPLEX [Toxoplasma gondii RH]\n",
      " 1QK4_B TOXOPLASMA GONDII HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "IMP COMPLEX [Toxoplasma gondii RH]\n",
      " 1QK4_C TOXOPLASMA GONDII HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "IMP COMPLEX [Toxoplasma gondii RH]\n",
      " 1QK4_D TOXOPLASMA GONDII HYPOXANTHINE-GUANINE PHOSPHORIBOSYLTRANSFERASE \n",
      "IMP COMPLEX [Toxoplasma gondii RH]\n",
      "Length=233\n",
      "\n",
      " Score = 65.5 bits (158),  Expect = 3e-12, Method: Compositional matrix adjust.\n",
      " Identities = 50/161 (31%), Positives = 83/161 (52%), Gaps = 13/161 (8%)\n",
      "\n",
      "Query  86   LIIISVLKGSYIFTSDFIRYLG--------DCGLPHVVD-FVRLASYNSGTKSTGQISML  136\n",
      "            L II +LKGS  F +  I YL         +  +P   + +VRL SY +   STGQ+++L\n",
      "Sbjct  75   LHIICILKGSRGFFNLLIDYLATIQKYSGRESSVPPFFEHYVRLKSYQNDN-STGQLTVL  133\n",
      "\n",
      "Query  137  AGLRFENLRGKHVLIVEDVCDSGRTLRFLRDYIMEKFQPKSIKTLVMVNKEQAARKVDFD  196\n",
      "            +       R KHVLIVED+ D+G TL    +  ++   PKS++   +V K +  R     \n",
      "Sbjct  134  SD-DLSIFRDKHVLIVEDIVDTGFTLTEFGER-LKAVGPKSMRIATLVEK-RTDRSNSLK  190\n",
      "\n",
      "Query  197  PEYFCLAGPNKYIVGYGFEVNDRYRDLRHILILRDGEATRY  237\n",
      "             ++   +  + +IVG  ++ N+ +RD  H+ +L D    ++\n",
      "Sbjct  191  GDFVGFSIEDVWIVGCCYDFNEMFRDFDHVAVLSDAARKKF  231\n",
      "\n",
      "\n",
      "  Database: PDB protein database\n",
      "    Posted date:  May 23, 2023  5:32 AM\n",
      "  Number of letters in database: 41,078,466\n",
      "  Number of sequences in database:  149,544\n",
      "\n",
      "Lambda      K        H\n",
      "   0.322    0.138    0.402 \n",
      "Gapped\n",
      "Lambda      K        H\n",
      "   0.267   0.0410    0.140 \n",
      "Matrix: BLOSUM62\n",
      "Gap Penalties: Existence: 11, Extension: 1\n",
      "Number of Sequences: 149544\n",
      "Number of Hits to DB: 112080\n",
      "Number of extensions: 164\n",
      "Number of successful extensions: 26\n",
      "Number of sequences better than 100: 11\n",
      "Number of HSP's better than 100 without gapping: 0\n",
      "Number of HSP's gapped: 18\n",
      "Number of HSP's successfully gapped: 11\n",
      "Length of query: 241\n",
      "Length of database: 41078466\n",
      "Length adjustment: 103\n",
      "Effective length of query: 138\n",
      "Effective length of database: 25675434\n",
      "Effective search space: 3543209892\n",
      "Effective search space used: 3543209892\n",
      "T: 19\n",
      "A: 40\n",
      "X1: 16 (7.4 bits)\n",
      "X2: 38 (14.6 bits)\n",
      "X3: 64 (24.7 bits)\n",
      "S1: 41 (20.4 bits)\n",
      "S2: 54 (25.4 bits)\n",
      "ka-blk-alpha gapped: 1.9\n",
      "ka-blk-alpha ungapped: 0.7916\n",
      "ka-blk-alpha_v gapped: 42.6028\n",
      "ka-blk-alpha_v ungapped: 4.96466\n",
      "ka-blk-sigma gapped: 43.6362\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from Bio import SeqIO\n",
    "from Bio.Blast import NCBIWWW, NCBIXML\n",
    "\n",
    "# Realiza a leitura do arquivo consulta que contém nossa sequência alvo\n",
    "fasta = SeqIO.read(\"query_2.fasta\", format = 'fasta')\n",
    "\n",
    "# Realizando uma pesquisa BLASTP com a sequência de entrada\n",
    "result = NCBIWWW.qblast(\"blastp\", \"pdb\", fasta.seq, format_type = 'Text')\n",
    "\n",
    "print(result.read())"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c6740c25-7e4c-41bf-a879-b7cfa5b8d086",
   "metadata": {},
   "source": [
    "Note que há três colunas, sendo: bitscore, E-value e Identidade. \n",
    "\n",
    "O valor de bitscore corresponde a significância estatística para cada alinhamento, sendo assim, quanto maior for o valor maior será a significância. O e-value nos indica se o alinhamento foi feito de forma aleatória com outra sequências, assim, quanto menor ou mais próximo de 0 for o valor, significa que as chances daquele alinhamento ter sido feito com uma sequência aleatória são pequenas. A identidade é o valor em porcentagem da correspondência entre os aminoácidos ou nucleotídeos (neste exemplo aminoácidos). Todos esses três parâmetros são essenciais para realizarmos inferências evolutivas e, assim, dizer qual sequência é homóloga ou não a nossa sequência alvo.\n",
    "\n",
    "Assim, iremos selecionar a sequência da proteína **7SCR**, pois ela possui os melhores valores para essas 3 variáveis citadas acima (bitscore=254, e-value=2e-88, identidade=55%).\n",
    "\n",
    "Ao pesquisarmos pela proteína de código **7SCR** no Protein Data Bank (https://www.rcsb.org/structure/7SCR), notamos que é uma enzima cristalografada pelo método de difração de raio-X. Trata-se da Hipoxantina-guanina Fosforribosil Transferase de _Trypanosoma brucei brucei_, uma enzima que pertecence a família da PRTases (assim como nosso alvo XPRT) e pertencente a um organismo parasiátario da família Trypanosomatidae assim como nosso organismo alvo, _Leishmania donovani_.\n",
    "\n",
    "Observamos também que a resolução desta proteína é de 2.12 Å (agntrons), onde quanto menor for esse valor, melhor será a resolução. Ela possui 4 cadeias (A, B, C e D), sendo que o blast apontou a cadeia A como melhor alinhamento. Observe também que esta enzima está o complexada com um nuclesídeo inibidor.  "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "89fdda97-9625-443d-9d09-53f4d279d4c0",
   "metadata": {},
   "source": [
    "Agora que já sabemos qual será o nosso modelo (7SCR) e considerando que você já instalou o Modeller versão 10.4, então iniciaremos a modelagem."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "d8dcdf29-2857-472d-9216-a45bac4052bd",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "--2023-05-25 18:35:50--  https://files.rcsb.org/download/7SCR.pdb\n",
      "Resolvendo files.rcsb.org (files.rcsb.org)... 128.6.158.70\n",
      "Conectando-se a files.rcsb.org (files.rcsb.org)|128.6.158.70|:443... conectado.\n",
      "A requisição HTTP foi enviada, aguardando resposta... 200 OK\n",
      "Tamanho: não especificada [application/octet-stream]\n",
      "Salvando em: “7SCR.pdb”\n",
      "\n",
      "     0K .......... .......... .......... .......... .......... 82,3K\n",
      "    50K .......... .......... .......... .......... ..........  233K\n",
      "   100K .......... .......... .......... .......... .......... 11,5M\n",
      "   150K .......... .......... .......... .......... .......... 83,0K\n",
      "   200K .......... .......... .......... .......... .......... 65,4M\n",
      "   250K .......... .......... .......... .......... ..........  166K\n",
      "   300K .......... .......... .......... .......... ..........  161K\n",
      "   350K .......... .......... .......... .......... ..........  164K\n",
      "   400K .......... .......... .......... .......... .......... 96,5K\n",
      "   450K .......... .......... .......... .......... .......... 52,4M\n",
      "   500K .......... .......... .......... .......... .......... 36,4M\n",
      "   550K .......... .......... .......... .......... ..........  238K\n",
      "   600K .......... .......... .......... .......... ..........  168K\n",
      "   650K .......... .......... .......... .......... ..........  163K\n",
      "   700K .......... .......... .......... .......... ..........  122K\n",
      "   750K .......... .......... .......... .......... .......... 54,4K\n",
      "   800K .......... .......... .......... .......... .......... 50,6M\n",
      "   850K .......... .......... .......... .......... ..........  166K\n",
      "   900K .......... .......... .......... .......... ..........  244K\n",
      "   950K .......... .......... .......... .......... ..........  246K\n",
      "  1000K ....                                                    632K=5,7s\n",
      "\n",
      "2023-05-25 18:35:57 (175 KB/s) - “7SCR.pdb” salvo [1029024]\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Realize o download da estrutura PDB 7SCR executando essa célula\n",
    "\n",
    "import os\n",
    "\n",
    "os.system(\"wget https://files.rcsb.org/download/7SCR.pdb\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cdd8af01-a41f-4e52-beaf-850e0887f3e4",
   "metadata": {},
   "source": [
    "Feito o download poderemos realizar o alinhamento entre as sequências (alvo x molde)."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0f53085d-cc3b-4636-a1ef-e33f7e9c1a6a",
   "metadata": {},
   "source": [
    "#### Alinhamento entre as sequências"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "55f820bf-a7bd-44e8-bd8c-367fd126f5c6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkapsa__637W> No residue topology library is in memory.\n",
      "              Better radii would be used if topology.read() is called first.\n",
      "iup2crm_280W> No topology library in memory or assigning a BLK residue.\n",
      "              Default CHARMM atom type assigned:  N -->  N\n",
      "              This message is written only for the first such atom.\n",
      "\n",
      "Pairwise dynamic programming alignment (ALIGN2D):\n",
      "  Residue-residue metric   : $(LIB)/as1.sim.mat\n",
      "  Diagonal                 :          100\n",
      "  Overhang                 :            0\n",
      "  Maximal gap length       :           50\n",
      "  Local alignment          :            F\n",
      "  MATRIX_OFFSET (local aln):       0.0000\n",
      "  FIX_OFFSETS              :      0.0    -1.0    -2.0    -3.0    -4.0\n",
      "  N_SUBOPT                 :            0\n",
      "  SUBOPT_OFFSET            :       0.0000\n",
      "  Alignment block          :            1\n",
      "  Gap introduction penalty :    -100.0000\n",
      "  Gap extension penalty    :       0.0000\n",
      "  Gap diagonal penalty     :       0.0000\n",
      "  Structure gap penalties  :    3.500   3.500   3.500   0.200   4.000   6.500   2.000   0.000\n",
      "  Break-break bonus        :   10000.0000\n",
      "  Length of alignment      :          241\n",
      "  Score                    :  154914.5312\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "\n",
    "env = Environ()\n",
    "aln = Alignment(env)\n",
    "mdl = Model(env, file='7SCR', model_segment=('FIRST:A','LAST:A'))\n",
    "# a variável \"file\" recebe como valor o código PDB da proteína molde\n",
    "aln.append_model(mdl, align_codes='7SCRA', atom_files='7SCR.pdb')\n",
    "aln.append(file='XpRT.ali', align_codes='XpRT')\n",
    "aln.align2d(max_gap_length=50)\n",
    "aln.write(file='XpRT-7SCRA.ali', alignment_format='PIR')\n",
    "aln.write(file='XpRT-7SCRA.pap', alignment_format='PAP')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4fecf725-b2f2-4dc7-a8e7-663b4f3c3a0e",
   "metadata": {},
   "source": [
    "#### Construção do modelo"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0349dca5-eba2-489f-9467-1191562e7f6a",
   "metadata": {},
   "source": [
    "Agora podemos construir nossa proteína a partir do nosso molde cristalográfico (PDB: 7SCR)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "88c8ad5e-ba6c-49de-a957-678218b08dff",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    122     1  116   130      S     Y    9.470\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    21451    19743\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      ">> Model assessment by DOPE potential\n",
      "iatmcls_286W> MODEL atom not classified:  LEU:OXT  LEU\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1899    1899\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19743   19743\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):   341149\n",
      "Dynamic pairs routine                             : 1, NATM x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        1    9999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :   15.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       F       F       F       T\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :      -26286.7246\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "DOPE score               : -26286.724609\n",
      ">> Model assessment by GA341 potential\n",
      "\n",
      "Surface library          : /home/lucas-sci/anaconda3/lib/modeller-10.4/modlib/surf5.de\n",
      "Pair library             : /home/lucas-sci/anaconda3/lib/modeller-10.4/modlib/pair9.de\n",
      "Chain identifier         : _\n",
      "% sequence identity      :    56.743999\n",
      "Sequence length          :          241\n",
      "Compactness              :     0.146090\n",
      "Native energy (pair)     :  -100.108575\n",
      "Native energy (surface)  :   -12.194101\n",
      "Native energy (combined) :    -5.350784\n",
      "Z score (pair)           :    -6.839611\n",
      "Z score (surface)        :    -6.491686\n",
      "Z score (combined)       :    -8.878672\n",
      "GA341 score              :     1.000000\n",
      "\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      241\n",
      "Number of all, selected real atoms                :     1899    1899\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    19743   19743\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     3954\n",
      "Dynamic pairs routine                             : 2, NATM x NATM cell sorting\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1658.9026\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :    1937       0      0   0.006   0.006      17.078       1.000\n",
      " 2 Bond angle potential               :    2617       0     10   2.081   2.081      223.43       1.000\n",
      " 3 Stereochemical cosine torsion poten:    1212       0     51  50.438  50.438      478.05       1.000\n",
      " 4 Stereochemical improper torsion pot:     801       0      1   1.204   1.204      27.727       1.000\n",
      " 5 Soft-sphere overlap restraints     :    3954       0      0   0.002   0.002      2.8706       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :    4073       1      8   0.258   0.258      152.12       1.000\n",
      "10 Distance restraints 2 (N-O)        :    4157       2     17   0.307   0.307      209.74       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:     240       1      8   5.776   5.776      94.411       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:     204       0      0  69.221  69.221      39.984       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:     155       0      1  57.950  57.950      44.461       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      65       0      0  69.977  69.977      42.759       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:      31       0      0  88.659  88.659      21.093       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :    2751       0      0   0.392   0.392      72.503       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:     239      26     30  30.540  60.471      151.04       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :    1261       0      3   0.669   0.669      81.640       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: XpRT.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   24506.8594\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  14234 162L  92L N   O    1226  691   12.73   10.77    1.96    4.53   10.77    1.96    4.53\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6571   4T   5H C   N      29   31  -80.37  -67.60   68.37    5.61  -63.20  116.41   13.56\n",
      "    1          5H   5H N   CA     31   32   72.84  140.00                  -42.30\n",
      "    2   6626  59D  60A C   N     453  455   68.63  -68.20  142.74   13.31  -62.50 -173.13   28.84\n",
      "    2         60A  60A N   CA    455  456 -174.04  145.30                  -40.90\n",
      "    3   6627  60A  61R C   N     458  460 -158.02 -125.20   88.75    3.46  -63.00  134.96   24.15\n",
      "    3         61R  61R N   CA    460  461 -136.94  140.60                  -41.10\n",
      "    4   6628  61R  62L C   N     469  471  -64.65  -70.70   34.58    2.40  -63.50  148.76   20.59\n",
      "    4         62L  62L N   CA    471  472  107.55  141.60                  -41.20\n",
      "    5   6629  62L  63S C   N     477  479 -149.71 -136.60   45.23    1.98  -64.10  156.08   17.49\n",
      "    5         63S  63S N   CA    479  480 -165.51  151.20                  -35.00\n",
      "    6   6631  64P  65S C   N     490  492  118.85 -136.60  113.66    3.56  -64.10 -140.81   28.28\n",
      "    6         65S  65S N   CA    492  493 -164.22  151.20                  -35.00\n",
      "    7   6632  65S  66T C   N     496  498  -84.48  -78.10   50.46    1.96  -63.20  119.95   16.80\n",
      "    7         66T  66T N   CA    498  499 -160.15  149.80                  -42.10\n",
      "    8   6633  66T  67A C   N     503  505  -64.66  -68.20    3.55    0.29  -62.50  174.07   28.63\n",
      "    8         67A  67A N   CA    505  506  145.04  145.30                  -40.90\n",
      "    9   6634  67A  68E C   N     508  510 -119.28 -117.80   22.21    1.09  -63.60  170.11   25.83\n",
      "    9         68E  68E N   CA    510  511  158.96  136.80                  -40.30\n",
      "   10   6659  92L  93K C   N     690  692 -113.44  -62.90   57.78   10.09  -62.90   57.78   10.09\n",
      "   10         93K  93K N   CA    692  693  -68.81  -40.80                  -40.80\n",
      "   11   6688 121A 122S C   N     922  924   82.89  -64.10  149.82   17.98 -136.60 -158.20    6.57\n",
      "   11        122S 122S N   CA    924  925  -63.96  -35.00                  151.20\n",
      "   12   6689 122S 123Y C   N     928  930 -149.91 -124.30   26.13    2.09   55.90  178.92   22.05\n",
      "   12        123Y 123Y N   CA    930  931  130.27  135.40                   39.50\n",
      "   13   6691 124N 125S C   N     948  950  162.27 -136.60   61.53    2.26  -64.10 -146.30   25.12\n",
      "   13        125S 125S N   CA    950  951  158.23  151.20                  -35.00\n",
      "   14   6692 125S 126G C   N     954  956   60.97   78.70   24.09    0.58   82.20  159.71    8.81\n",
      "   14        126G 126G N   CA    956  957 -149.79 -166.10                    8.50\n",
      "   15   6693 126G 127T C   N     958  960 -118.75 -124.80    8.54    0.28  -63.20 -172.04   20.94\n",
      "   15        127T 127T N   CA    960  961  137.46  143.50                  -42.10\n",
      "   16   6694 127T 128K C   N     965  967  -69.88  -70.20    4.28    0.30  -62.90  177.07   22.62\n",
      "   16        128K 128K N   CA    967  968  136.13  140.40                  -40.80\n",
      "   17   6695 128K 129S C   N     974  976  -87.34  -72.40   24.96    1.14  -64.10  154.36   12.71\n",
      "   17        129S 129S N   CA    976  977  172.40  152.40                  -35.00\n",
      "   18   6696 129S 130T C   N     980  982  -94.06  -78.10   22.67    1.46  -63.20  178.49   20.99\n",
      "   18        130T 130T N   CA    982  983  133.70  149.80                  -42.10\n",
      "   19   6698 131G 132Q C   N     991  993  -68.80  -73.00    5.52    0.32  -63.80  177.50   25.84\n",
      "   19        132Q 132Q N   CA    993  994  137.13  140.70                  -40.30\n",
      "   20   6700 133I 134S C   N    1008 1010 -103.00  -72.40   55.35    2.53  -64.10  132.32   12.29\n",
      "   20        134S 134S N   CA   1010 1011 -161.47  152.40                  -35.00\n",
      "   21   6701 134S 135M C   N    1014 1016  -62.44  -73.00   37.37    2.38  -63.40  147.66   22.12\n",
      "   21        135M 135M N   CA   1016 1017  107.15  143.00                  -40.50\n",
      "   22   6702 135M 136L C   N    1022 1024 -144.20 -108.50   36.38    1.66  -63.50 -163.42   32.29\n",
      "   22        136L 136L N   CA   1024 1025  139.54  132.50                  -41.20\n",
      "   23   6703 136L 137A C   N    1030 1032 -141.80 -134.00   16.26    0.67  -62.50  176.64   33.19\n",
      "   23        137A 137A N   CA   1032 1033  161.27  147.00                  -40.90\n",
      "   24   6704 137A 138G C   N    1035 1037 -149.34 -167.20   81.89    2.94  -80.20  105.29    7.27\n",
      "   24        138G 138G N   CA   1037 1038   94.68  174.60                  174.10\n",
      "   25   6723 156C 157D C   N    1188 1190  -86.90  -63.30   46.44    7.24  -63.30   46.44    7.24\n",
      "   25        157D 157D N   CA   1190 1191  -80.00  -40.00                  -40.00\n",
      "   26   6806 239A 240K C   N    1880 1882  -66.67  -62.90   14.11    1.67  -70.20  167.64   12.06\n",
      "   26        240K 240K N   CA   1882 1883  -27.20  -40.80                  140.40\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0   15   35  100  110  167  171  221  255  294  329\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf     DOPE score    GA341 score\n",
      "----------------------------------------------------------------------\n",
      "XpRT.B99990001.pdb            1658.90259   -26286.72461        1.00000\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from modeller import *\n",
    "from modeller.automodel import *\n",
    "\n",
    "env = Environ()\n",
    "a = AutoModel(env, alnfile='XpRT-7SCRA.ali',\n",
    "              knowns='7SCRA', sequence='XpRT',\n",
    "              assess_methods=(assess.DOPE,\n",
    "                              #soap_protein_od.Scorer(),\n",
    "                              assess.GA341))\n",
    "a.starting_model = 1\n",
    "a.ending_model = 1\n",
    "# Será construído apenas um modelo, porém poderiam ser construído mais modelos\n",
    "a.make()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "41d853f1-fb0c-42d4-b3d6-0263260a5109",
   "metadata": {},
   "source": [
    "Construída a nossa proteínas podemos agora visualizar ela através da biblioteca NGLview."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "b9e8c8f4-1d6f-44ae-bdba-9764197fdb4c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "7c9ca161f719401abbb414c0a13a876e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": []
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "7245abb6ff254153b603ee87a7ad4de2",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import nglview as nv\n",
    "\n",
    "view = nv.show_file('XpRT.B99990001.pdb')\n",
    "# o arquivo PDB da estrutura será guardado nessa variável \"view\"\n",
    "\n",
    "view"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "53ab399a",
   "metadata": {},
   "source": [
    "Agora, a fim de comparar as estruturas, iremos abrir a proteína molde (PDB: 7SCR). Mas antes precisamos remover as cadeias B, C, D, E e F do arquivo _7SCR.pdb_ e então salvar em um novo aquivo. Desta forma, execute a seguinte célula."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "85dd8519",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.PDB import PDBParser, PDBIO\n",
    "import warnings\n",
    "\n",
    "warnings.simplefilter(\"ignore\")\n",
    "\n",
    "# Caminho para o arquivo PDB da proteína\n",
    "pdb_file = \"7SCR.pdb\"\n",
    "\n",
    "# Seleciona a cadeia (neste caso a cadeia que não utilizamos para modelar)\n",
    "chain_id = ['B','C','D','E','F']\n",
    "\n",
    "# Cria um objeto PDBParser\n",
    "parser = PDBParser()\n",
    "\n",
    "# Faz a leitura da estrutura a partir do arquivo PDB\n",
    "structure = parser.get_structure('proteina', pdb_file)\n",
    "\n",
    "# Remove a cadeia desejada\n",
    "for chains in chain_id:\n",
    "    model = structure[0]\n",
    "    model.detach_child(chains)\n",
    "\n",
    "# Salva a nova estrutura em um arquivo PDB\n",
    "output_file = 'new_7SCR.pdb'\n",
    "io = PDBIO()\n",
    "io.set_structure(structure)\n",
    "io.save(output_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "eedaa969",
   "metadata": {},
   "source": [
    "Removida as cadeias e deixando apenas a cadeia A, ao qual foi a que utilizamos para modelar a nossa sequencia alvo de XPRT, então podemos visualizar a estrutura cristalográfica executando a célula a seguir:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "cb8ec09f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "54af44c8298045e398ddef2d56bfc45a",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "NGLWidget()"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import nglview as nv\n",
    "\n",
    "view2 = nv.show_file('new_7SCR.pdb')\n",
    "# o arquivo PDB da estrutura será guardado nessa variável \"view\"\n",
    "\n",
    "view2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8acc972f",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
